
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />






  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>



    <style>


  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>






            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />


        <title>
            Clinical Research Informatics: a Decade-in-Review - PMC
        </title>




  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="ABCBC0C78F2310F31EC0C7004D3A4C58.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />

    <link rel="preconnect" href="https://www.google-analytics.com" />


        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />


    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ymi">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12020646/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Yearbook of Medical Informatics">
<meta name="citation_title" content="Clinical Research Informatics: a Decade-in-Review">
<meta name="citation_author" content="Christel Daniel">
<meta name="citation_author_institution" content="AP-HP, France">
<meta name="citation_author_institution" content="Sorbonne Université, INSERM UMR_S 1142, LIMICS, F-75006, Paris, France">
<meta name="citation_author" content="Peter J Embí">
<meta name="citation_author_institution" content="Vanderbilt University Medical Center, Department of Biomedical Informatics, Nashville, Tennessee, USA">
<meta name="citation_publication_date" content="2025 Apr 8">
<meta name="citation_volume" content="33">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="127">
<meta name="citation_doi" content="10.1055/s-0044-1800732">
<meta name="citation_pmid" content="40199298">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12020646/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12020646/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12020646/pdf/10-1055-s-0044-1800732.pdf">
<meta name="description" content="Background : Clinical Research Informatics (CRI) is a subspeciality of biomedical informatics that has substantially matured during the last decade. Advances in CRI have transformed the way clinical research is conducted. In recent years, there has ...">
<meta name="og:title" content="Clinical Research Informatics: a Decade-in-Review">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Background : Clinical Research Informatics (CRI) is a subspeciality of biomedical informatics that has substantially matured during the last decade. Advances in CRI have transformed the way clinical research is conducted. In recent years, there has ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12020646/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">



    </head>
    <body >

    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>





<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>






















        <button
            type="button"


    class="usa-accordion__button usa-banner__button












           "
    aria-expanded="false"
    aria-controls="gov-banner-default"

    data-testid="storybook-django-banner"

    >




                    <span class="usa-banner__button-text">Here's how you know</span>





            </button>



            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>








<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->





















        <button
            type="button"


    class="usa-menu-btn ncbi-header__hamburger-button












           "


    aria-label="Show menu"
    data-testid="navMenuButton"

    >




                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>





            </button>





                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">

                        <!-- Desktop search button -->





















        <button
            type="button"


    class="usa-button











           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >








    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >

        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search





            </button>





                    <!-- Desktop login dropdown -->

                        <div class="ncbi-header__login-dropdown">





















        <button
            type="button"


    class="usa-button











           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >








    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >

        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->




    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>

        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->




    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>

        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>







            </button>




                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>

                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->










    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >



            Dashboard



    </a>


                                    </li>

                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->










    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >



            Publications



    </a>


                                    </li>

                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->










    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >



            Account settings



    </a>


                                    </li>

                                <li class="usa-nav__submenu-item">





















        <button
            type="button"


    class="usa-button











           usa-button--outline ncbi-header__login-dropdown-logout-button
           "



    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >



        Log out




            </button>



                                </li>
                            </ul>
                        </div>

                </div>

        </div>
    </div>

    <!-- Search panel -->

        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"

                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"

                               placeholder="Search NCBI"

                           type="search"
                           value="" />





















        <button
            type="submit"


    class="usa-button












           "





    >




                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>





            </button>



                </form>


            </div>
        </div>


    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->





















        <button
            type="button"


    class="usa-nav__close ncbi-nav__close-button












           "


    aria-label="Close navigation menu"
    data-testid="navCloseButton"

    >




            <img src="/static/img/usa-icons/close.svg" alt="Close" />





            </button>





            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"

                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"

                           placeholder="Search NCBI"

                       name="term" />






















        <button
            type="submit"


    class="usa-button












           "





    >




                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />





            </button>



            </form>




        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->



            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">

                        <li class="usa-nav__primary-item">










    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >



            Dashboard



    </a>


                        </li>

                        <li class="usa-nav__primary-item">










    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >



            Publications



    </a>


                        </li>

                        <li class="usa-nav__primary-item">










    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >



            Account settings



    </a>


                        </li>

                </ul>
            </div>























        <button
            type="button"


    class="usa-button











           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "



    data-testid="mobileLoginButton"
    data-mobile-login-button
    >



        Log in




            </button>



    </nav>
</header>





<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">



<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://pmc.ncbi.nlm.nih.gov/search/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>

                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>

                    </li>
                    <li>

                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>

                    </li>
                </ul>
            </div>
        </div>
    </div>
</section>








  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12020646">







<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>

                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1055/s-0044-1800732"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>


                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/10-1055-s-0044-1800732.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>

                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12020646%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>

                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12020646/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12020646/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>

                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>


<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12020646/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >


                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ymi.png" alt="Yearbook of Medical Informatics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Yearbook of Medical Informatics" title="Link to Yearbook of Medical Informatics" shape="default" href="https://www.thieme-connect.com/DOI/DOI?10.1055/s-00034612" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Yearb Med Inform</button></div>. 2025 Apr 8;33(1):127–142. doi: <a href="https://doi.org/10.1055/s-0044-1800732" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1055/s-0044-1800732</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://pmc.ncbi.nlm.nih.gov/search/?term=%22Yearb%20Med%20Inform%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yearb%20Med%20Inform%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Yearb%20Med%20Inform%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Yearb%20Med%20Inform%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Clinical Research Informatics: a Decade-in-Review</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Daniel%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Christel Daniel</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Christel Daniel</span></h3>
<div class="p">
<sup>1</sup>AP-HP, France</div>
<div class="p">
<sup>2</sup>Sorbonne Université, INSERM UMR_S 1142, LIMICS, F-75006, Paris, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Daniel%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christel Daniel</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Emb%C3%AD%20PJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Peter J Embí</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Peter J Embí</span></h3>
<div class="p">
<sup>3</sup>Vanderbilt University Medical Center, Department of Biomedical Informatics, Nashville, Tennessee, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Emb%C3%AD%20PJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter J Embí</span></a>
</div>
</div>
<sup>3</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="AFdaniel-1">
<sup>1</sup>AP-HP, France</div>
<div id="AFdaniel-2">
<sup>2</sup>Sorbonne Université, INSERM UMR_S 1142, LIMICS, F-75006, Paris, France</div>
<div id="AFdaniel-3">
<sup>3</sup>Vanderbilt University Medical Center, Department of Biomedical Informatics, Nashville, Tennessee, USA</div>
<div class="author-notes p"><div class="fn" id="COdaniel-1">
<sup>✉</sup><p class="display-inline"><strong>Correspondence to: </strong>Christel Daniel Sorbonne Université, INSERM UMR_S 1142, LIMICS, F-75006, Paris, France, Email: <span>christel.daniel@aphp.fr</span></p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Collection date 2024 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>
The Author(s). This is an open access article published by Thieme under the terms of the Creative Commons Attribution License, permitting unrestricted use, distribution, and reproduction so long as the original work is properly cited. (
<a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://creativecommons.org/licenses/by/4.0/</a>
)
</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use,
distribution, and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12020646  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40199298/" class="usa-link">40199298</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Summary</h2>
<p><strong>Background</strong>
: Clinical Research Informatics (CRI) is a subspeciality of biomedical informatics that has substantially matured during the last decade. Advances in CRI have transformed the way clinical research is conducted. In recent years, there has been growing interest in CRI, as reflected by a vast and expanding scientific literature focused on the topic. The main objectives of this review are: 1) to provide an overview of the evolving definition and scope of this biomedical informatics subspecialty over the past 10 years; 2) to highlight major contributions to the field during the past decade; and 3) to provide insights about more recent CRI research trends and perspectives.
</p>
<p><strong>Methods</strong>
: We adopted a modified thematic review approach focused on understanding the evolution and current status of the CRI field based on literature sources identified through two complementary review processes (AMIA CRI year-in-review/IMIA Yearbook of Medical Informatics) conducted annually during the last decade.
</p>
<p><strong>Results</strong>
: More than 1,500 potentially relevant publications were considered, and 205 sources were included in the final review. The review identified key publications defining the scope of CRI and/or capturing its evolution over time as illustrated by impactful tools and methods in different categories of CRI focus. The review also revealed current topics of interest in CRI and prevailing research trends.
</p>
<p><strong>Conclusion</strong>
: This scoping review provides an overview of a decade of research in CRI, highlighting major changes in the core CRI discoveries as well as increasingly impactful methods and tools that have bridged the principles-to-practice gap. Practical CRI solutions as well as examples of CRI-enabled large-scale, multi-organizational and/or multi-national research projects demonstrate the maturity of the field. Despite the progress demonstrated, some topics remain challenging, highlighting the need for ongoing CRI development and research, including the need of more rigorous evaluations of CRI solutions and further formalization and maturation of CRI services and capabilities across the research enterprise.
</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> Clinical Research Informatics, Biomedical Research, Clinical Trials, Informatics, Literature Review</p></section></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>
Clinical research, whether interventional (
<em>e.g.</em>
, clinical trials) or observational (
<em>e.g.</em>
, quasi-experimental studies, outcomes research), encompasses research conducted on humans or on materials of human origin (
<em>e.g.</em>
, real-world data), and is critical to advancing medical science and public health. Conducting such research is a complex, information-intensive effort, involving multiple actors, workflows, processes, and resources. In this context emerged over 15 years ago the field of Clinical Research Informatics (CRI), a biomedical informatics sub-discipline focused on clinical research that was first defined in 2009 [
<a href="#ORdaniel-1" class="usa-link" aria-describedby="ORdaniel-1">1</a>
]. More recently, several literature reviews have been conducted to further frame and refine the topics of CRI in general [
<a href="#ORdaniel-2" class="usa-link" aria-describedby="ORdaniel-2">2</a>
,
<a href="#ORdaniel-3" class="usa-link" aria-describedby="ORdaniel-3">3</a>
]
</p>
<p>The focus of this review is to provide an updated practical and informative overview of the CRI field based on examples of impactful papers selected during the last decade through two regular review processes: the annual “CRI year-in-review” presented by at the Informatics Summit of the American Medical Informatics Association (AMIA year-in-review), and the periodic CRI reviews in the IMIA Yearbook of Medical Informatics (IMIA Yearbook).</p>
<p>Our objectives are threefold:</p>
<ul class="list" style="list-style-type:disc">
<li><p>to provide an overview of the evolving scope and categories of focus for this subspecialty of biomedical informatics over the past 10 years;</p></li>
<li><p>to highlight major contributions to the CRI field during the last decade in each category and identify efforts by the CRI community to address practical informatics needs for advancing clinical research;</p></li>
<li><p>
to provide insights on more recent (
<em>i.e.</em>
, past two years) contributions in CRI and identify current research trends and perspectives for the CRI field.
</p></li>
</ul>
<p>In the sections that follow, we first describe source selection and information extraction. We then highlight major contributions according to the three dimensions presented above (CRI definition, major CRI contributions, and highlight recent research trends and perspectives).</p></section><section id="sec2"><h2 class="pmc_sec_title">2. Methods</h2>
<p>The authors adopted a modified thematic review approach based, in a pragmatic way, on the analysis of the literature sources resulting from two review processes (AMIA CRI year-in-review/IMIA Yearbook) conducted annually since 2013.</p>
<section id="sec3"><h3 class="pmc_sec_title">2.1. Research question and scoping questions</h3>
<p>The core research question of this review focuses on practically relevant aspects of CRI supporting various steps of the life cycle of clinical research. The authors agreed to consider the following scoping questions: “How has the definition and scope of CRI evolved along time?”; “What are examples of CRI tools and methods that have reached sufficient maturity to support clinical research?”; and “What are the current trends and major challenges for CRI research today?”</p></section><section id="sec4"><h3 class="pmc_sec_title">2.2. Information sources, search strategy and paper selection</h3>
<p><a href="#TBdaniel-1" class="usa-link">Table 1</a>
summarizes the search strategies used by: (a) Embi in preparation of the “CRI year in review” presented annually at the AMIA Informatics Summit, and (b) by the editors of the CRI section of the IMIA Yearbook.
<a href="#FIdaniel-1" class="usa-link">Figure 1</a>
summarizes how the authors identified and evaluated the literature sources shortlisted by both AMIA year-in-review and IMIA Yearbook processes, and then applied those approaches to this review.
</p>
<section class="tw xbox font-sm" id="TBdaniel-1"><h4 class="obj_head">Table 1. Search strategy and paper selection in both AMIA CRI year-in-review and IMIA Yearbook processes.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" rules="all">
<colgroup align="left" span="2">
<col align="left" width="20.0%" span="1">
<col align="left" width="80.0%" span="1">
</colgroup>
<tbody>
<tr><td align="left" colspan="2" valign="top" rowspan="1">
<strong>1 – Search strategy</strong>
</td></tr>
<tr><td align="left" colspan="2" valign="top" rowspan="1">MEDLINE query via PubMed</td></tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">(a) AMIA CRI year-in-review</td>
<td align="left" valign="top" colspan="1" rowspan="1">(b) IMIA Yearbook CRI section</td>
</tr>
<tr>
<td align="left" rowspan="4" valign="top" colspan="1"> <br>(a) Initial search by MeSH terms and keywords, including:“Biomedical Research”[MeSH] AND “Informatics”[MeSH] NOT (“computational biology”[MeSH] OR “genetics”[MeSH]) (b) Limit to (e.g., English, with Abstracts, etc.)<br>Additional articles identified using date restrictions and keyword searches with terms such as: Clinical Trials, Clinical Research, Informatics, Translational, Data Warehouse, Research Registries, Recruitment.<br>Recommendations from colleagues.</td>
<td align="left" valign="top" colspan="1" rowspan="1">Two conceptual axes: clinical research and informatics</td>
</tr>
<tr><td align="left" valign="top" colspan="1" rowspan="1">((Biomedical Research[MAJR] OR „Biomedical Research“[TIAB] OR „Clinical research“[TIAB] OR „Medical research“[TIAB] OR Nursing Research[MAJR] OR „Nursing Research“[TIAB] OR „Pharmacovigilance“[TIAB] OR „Patient Selection“[TIAB] OR „phenotyping“[TIAB] OR „genotype-phenotype associations“[TIAB] OR Epidemiologic Research Design[MAJR] OR „Epidemiological Monitoring“[TIAB] OR Evaluation Studies as Topic[MAJR] OR Clinical Studies as Topic[MAJR] OR Multicenter Studies as Topic[MAJR] OR Big data [MAJR] OR „Feasibility Studies“[TIAB] OR “clinical research informatics” [TIAB] OR “eligibility criteria” [TIAB] OR “feasibility criteria” [TIAB] OR “cohort selection” [TIAB] OR “patient recruitment” [TIAB] OR “clinical trial eligibility screening” [TIAB] OR “eligibility determination” [TIAB] OR “patient-trial matching” [TIAB] OR “protocol feasibility” [TIAB] OR “real world evidence” [TIAB] OR data science[MAJR] OR „data science“[TIAB] OR Clinical trial protocols as topic[MAJR])</td></tr>
<tr><td align="left" valign="top" colspan="1" rowspan="1">AND (medical informatics[MAJR] OR „medical informatics“[TIAB] OR „clinical informatics“[TIAB] OR „medical computer science“[TIAB] OR „medical information science“[TIAB] OR Informatics[MAJR] OR „Informatics“[TIAB] OR „electronic healthcare record“[all fields] OR „electronic health record“[all fields] OR „electronic medical record“[all fields] OR „electronic patient record“[all fields] OR „personal health record“[all fields] OR “big data“[TIAB] OR “real world data” [TIAB]<br>OR (Data Warehousing[MeSH] OR „Data Warehousing“[TIAB] OR „Data Warehouse“[TIAB] OR big data[MeSH] OR „big data“[TIAB]))</td></tr>
<tr><td align="left" valign="top" colspan="1" rowspan="1">AND (yyyy[DP])<br>AND English[lang] AND hasabstract[text]</td></tr>
<tr><td align="left" colspan="2" valign="top" rowspan="1">2 – Paper selection</td></tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">From the total identified, Embi selected a sampling of impactful papers to be presented in detail, and others to receive brief mentions.</td>
<td align="left" valign="top" colspan="1" rowspan="1">Section editors apply exclusion criteria to the original set of retrieved references to select papers in the scope of CRI and blindly evaluate their contribution based on titles and abstracts. The resulting highly contributive papers are reviewed jointly by the section editors to select a consensual list of approximately 15 candidate best papers representative of all CRI categories. In conformance with the IMIA Yearbook process, these shortlisted papers are peer-reviewed by four IMIA Yearbook editors (the two section editors, and two editors in chief), and external reviewers in order to finally select four papers as best papers.</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/TBdaniel-1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="FIdaniel-1"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/de91/12020646/0eb6e4e20911/10-1055-s-0044-1800732-idaniel-1.jpg" loading="lazy" height="405" width="800" alt="Figure 1."></p>
<div class="p text-right font-secondary"><a href="figure/FIdaniel-1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>
Overview of the article selection in both AMIA CRI year-in-review and IMIA Yearbook processes, and as applied for purposes of this review (highlighted in the blue box).
</p></figcaption></figure></section><section id="sec5"><h3 class="pmc_sec_title">2.3. Final paper selection and analysis</h3>
<p>
The search strategy and paper selection described in the previous section resulted in a total of 345 (164 from the AMIA year-in-review, and 181 from the IMIA Yearbook) scientific publications, and of 327 after removal of duplicates (n=18). After additional literature sources according to the author's expertise and final selection, 205 documents remained on the shortlist included as references for this review paper. Consensus between the two reviewers, who are also the co-authors, was reached by discussion. As with the AMIA and IMIA processes on which this approach was based, the ultimate selection of impactful papers reflects the views of the authors, and is not meant an exhaustive or systematic review
<em>per se</em>
.
</p></section></section><section id="sec6"><h2 class="pmc_sec_title">3. Results</h2>
<section id="sec7"><h3 class="pmc_sec_title">3.1. CRI definition and categories</h3>
<section id="sec8"><h4 class="pmc_sec_title">3.1.1. CRI definition and scope</h4>
<p>
In 2009, Embi and Payne proposed a definition of clinical research informatics (CRI) as “the subdomain of biomedical informatics concerned with the development, application, and evaluation of theories, methods, and systems to optimize the design and conduct of clinical research and the analysis, interpretation, and dissemination of the information generated” [
<a href="#ORdaniel-1" class="usa-link" aria-describedby="ORdaniel-1">1</a>
,
<a href="#ORdaniel-4" class="usa-link" aria-describedby="ORdaniel-4">4</a>
] Defined as the intersection of the field of clinical research and the field of biomedical informatics, CRI focuses on the development and evaluation of tools and methods to support researchers in clinical research activities including study design, patient recruitment, data collection, integration and analysis. This is further illustrated in the conceptual framework of CRI proposed by Weng and Kahn (
<a href="#FIdaniel-2" class="usa-link">Figure 2</a>
) [
<a href="#ORdaniel-2" class="usa-link" aria-describedby="ORdaniel-2">2</a>
]
</p>
<figure class="fig xbox font-sm" id="FIdaniel-2"><h5 class="obj_head">Figure 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/de91/12020646/cf69de24ace2/10-1055-s-0044-1800732-idaniel-2.jpg" loading="lazy" height="570" width="800" alt="Figure 2."></p>
<div class="p text-right font-secondary"><a href="figure/FIdaniel-2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>
Clinical Research Informatics (CRI) conceptual framework [
<a href="#ORdaniel-2" class="usa-link" aria-describedby="ORdaniel-2">2</a>
].
</p></figcaption></figure><p>
Subsequently, Johnson proposed a search strategy using two conceptual axes defining CRI and sharpening the boundaries with closely related fields such as computational statistics and patient care informatics [
<a href="#ORdaniel-3" class="usa-link" aria-describedby="ORdaniel-3">3</a>
]
</p>
<p>
As noted by these authors and others, CRI approaches and capabilities have the potential to play a prominent role in supporting widespread advances in medicine, healthcare, and public health [
<a href="#ORdaniel-5" class="usa-link" aria-describedby="ORdaniel-5">5</a>
]
</p></section><section id="sec9"><h4 class="pmc_sec_title">3.1.2. Methodological pillars and categories</h4>
<p>
Progress by the community of CRI researchers and developers can be depicted as advancing along major pillars of the informatics sub-discipline such as data integration, information extraction, data linkage, data quality assessment, phenotyping, data and text mining, knowledge engineering, and health technology development and assessment (workflow support, human computer interaction, evaluation) in order to reinforce methods and tools supporting the different stages. The papers selected by the AMIA year-in-review and IMIA Yearbook processes have been classified into other sub-categories related to topics and/or steps of the clinical research lifecycle for which they provide solutions [
<a href="#ORdaniel-3" class="usa-link" aria-describedby="ORdaniel-3">3</a>
] .
<a href="#TBdaniel-2" class="usa-link">Table 2</a>
summarizes the classification of the final list of selected papers identified via the combined review processes utilized for the current review.
</p>
<section class="tw xbox font-sm" id="TBdaniel-2"><h5 class="obj_head">Table 2. Categorization and sub-categorization of the 205 selected papers.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" rules="all">
<colgroup align="left" span="6">
<col align="left" width="9.0%" span="1">
<col align="left" width="14.0%" span="1">
<col align="left" width="6.0%" span="1">
<col align="left" width="33.0%" span="1">
<col align="left" width="26.0%" span="1">
<col align="left" width="12.0%" span="1">
</colgroup>
<thead><tr>
<th align="left" colspan="3" valign="bottom" rowspan="1">Categories<br>(Acronym, label, number of papers)</th>
<th align="left" valign="bottom" colspan="1" rowspan="1">Sub categories</th>
<th align="left" valign="bottom" colspan="1" rowspan="1">Corresponding stage of the clinical research study life cycle</th>
<th align="left" valign="bottom" colspan="1" rowspan="1">Sources</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="3" valign="top" colspan="1">SD-PR-EXE</td>
<td align="left" rowspan="3" valign="top" colspan="1">Study design &amp; executionPatient recruitment</td>
<td align="left" rowspan="3" valign="top" colspan="1">n=43</td>
<td align="left" valign="top" colspan="1" rowspan="1">Study Design (new CT designs)</td>
<td align="left" valign="top" colspan="1" rowspan="1">CRI step 1: Analyzing Study Designs, feasibility studies</td>
<td align="left" rowspan="3" valign="top" colspan="1">
[
<a href="#ORdaniel-6" class="usa-link" aria-describedby="ORdaniel-6">6</a>
<a href="#ORdaniel-7" class="usa-link" aria-describedby="ORdaniel-7">7</a>
<a href="#ORdaniel-8" class="usa-link" aria-describedby="ORdaniel-8">8</a>
<a href="#ORdaniel-9" class="usa-link" aria-describedby="ORdaniel-9">9</a>
<a href="#ORdaniel-10" class="usa-link" aria-describedby="ORdaniel-10">10</a>
<a href="#ORdaniel-11" class="usa-link" aria-describedby="ORdaniel-11">11</a>
<a href="#ORdaniel-12" class="usa-link" aria-describedby="ORdaniel-12">12</a>
<a href="#ORdaniel-13" class="usa-link" aria-describedby="ORdaniel-13">13</a>
<a href="#ORdaniel-14" class="usa-link" aria-describedby="ORdaniel-14">14</a>
<a href="#ORdaniel-15" class="usa-link" aria-describedby="ORdaniel-15">15</a>
<a href="#ORdaniel-16" class="usa-link" aria-describedby="ORdaniel-16">16</a>
<a href="#ORdaniel-17" class="usa-link" aria-describedby="ORdaniel-17">17</a>
<a href="#ORdaniel-18" class="usa-link" aria-describedby="ORdaniel-18">18</a>
<a href="#ORdaniel-19" class="usa-link" aria-describedby="ORdaniel-19">19</a>
<a href="#ORdaniel-20" class="usa-link" aria-describedby="ORdaniel-20">20</a>
<a href="#ORdaniel-21" class="usa-link" aria-describedby="ORdaniel-21">21</a>
<a href="#ORdaniel-22" class="usa-link" aria-describedby="ORdaniel-22">22</a>
<a href="#ORdaniel-23" class="usa-link" aria-describedby="ORdaniel-23">23</a>
<a href="#ORdaniel-24" class="usa-link" aria-describedby="ORdaniel-24">24</a>
<a href="#ORdaniel-25" class="usa-link" aria-describedby="ORdaniel-25">25</a>
<a href="#ORdaniel-26" class="usa-link" aria-describedby="ORdaniel-26">26</a>
<a href="#ORdaniel-27" class="usa-link" aria-describedby="ORdaniel-27">27</a>
<a href="#ORdaniel-28" class="usa-link" aria-describedby="ORdaniel-28">28</a>
<a href="#ORdaniel-29" class="usa-link" aria-describedby="ORdaniel-29">29</a>
<a href="#ORdaniel-30" class="usa-link" aria-describedby="ORdaniel-30">30</a>
<a href="#ORdaniel-31" class="usa-link" aria-describedby="ORdaniel-31">31</a>
<a href="#ORdaniel-32" class="usa-link" aria-describedby="ORdaniel-32">32</a>
<a href="#ORdaniel-33" class="usa-link" aria-describedby="ORdaniel-33">33</a>
<a href="#ORdaniel-34" class="usa-link" aria-describedby="ORdaniel-34">34</a>
<a href="#ORdaniel-35" class="usa-link" aria-describedby="ORdaniel-35">35</a>
<a href="#ORdaniel-36" class="usa-link" aria-describedby="ORdaniel-36">36</a>
<a href="#ORdaniel-37" class="usa-link" aria-describedby="ORdaniel-37">37</a>
<a href="#ORdaniel-38" class="usa-link" aria-describedby="ORdaniel-38">38</a>
<a href="#ORdaniel-39" class="usa-link" aria-describedby="ORdaniel-39">39</a>
<a href="#ORdaniel-40" class="usa-link" aria-describedby="ORdaniel-40">40</a>
<a href="#ORdaniel-41" class="usa-link" aria-describedby="ORdaniel-41">41</a>
<a href="#ORdaniel-42" class="usa-link" aria-describedby="ORdaniel-42">42</a>
<a href="#ORdaniel-43" class="usa-link" aria-describedby="ORdaniel-43">43</a>
<a href="#ORdaniel-44" class="usa-link" aria-describedby="ORdaniel-44">44</a>
<a href="#ORdaniel-45" class="usa-link" aria-describedby="ORdaniel-45">45</a>
<a href="#ORdaniel-46" class="usa-link" aria-describedby="ORdaniel-46">46</a>
<a href="#ORdaniel-47" class="usa-link" aria-describedby="ORdaniel-47">47</a>
<a href="#ORdaniel-48" class="usa-link" aria-describedby="ORdaniel-48">48</a>
]
</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">Patient Recruitment</td>
<td align="left" valign="top" colspan="1" rowspan="1">CRI step 2: Getting Participants into Studies</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">Study Execution (CRI methods &amp; systems, collaborative workflow systems, usability and needs)</td>
<td align="left" rowspan="2" valign="top" colspan="1">CRI step 3: Executing Studies</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">PTF</td>
<td align="left" valign="top" colspan="1" rowspan="1">Large scale research platforms</td>
<td align="left" valign="top" colspan="1" rowspan="1">n=12</td>
<td align="left" valign="top" colspan="1" rowspan="1">Study Execution (large scale collaborative platforms, data networks)</td>
<td align="left" valign="top" colspan="1" rowspan="1">
[
<a href="#ORdaniel-49" class="usa-link" aria-describedby="ORdaniel-49">49</a>
<a href="#ORdaniel-50" class="usa-link" aria-describedby="ORdaniel-50">50</a>
<a href="#ORdaniel-51" class="usa-link" aria-describedby="ORdaniel-51">51</a>
<a href="#ORdaniel-52" class="usa-link" aria-describedby="ORdaniel-52">52</a>
<a href="#ORdaniel-53" class="usa-link" aria-describedby="ORdaniel-53">53</a>
<a href="#ORdaniel-54" class="usa-link" aria-describedby="ORdaniel-54">54</a>
<a href="#ORdaniel-55" class="usa-link" aria-describedby="ORdaniel-55">55</a>
<a href="#ORdaniel-56" class="usa-link" aria-describedby="ORdaniel-56">56</a>
<a href="#ORdaniel-57" class="usa-link" aria-describedby="ORdaniel-57">57</a>
<a href="#ORdaniel-58" class="usa-link" aria-describedby="ORdaniel-58">58</a>
<a href="#ORdaniel-59" class="usa-link" aria-describedby="ORdaniel-59">59</a>
<a href="#ORdaniel-60" class="usa-link" aria-describedby="ORdaniel-60">60</a>
]
</td>
</tr>
<tr>
<td align="left" rowspan="4" valign="top" colspan="1">DM</td>
<td align="left" rowspan="4" valign="top" colspan="1">Study data management</td>
<td align="left" rowspan="4" valign="top" colspan="1">n=59</td>
<td align="left" valign="top" colspan="1" rowspan="1">Managing Study Data (focus real-world data/phenotyping)<br>(n=17)</td>
<td align="left" rowspan="4" valign="top" colspan="1">CRI step 4: Managing Study Data</td>
<td align="left" valign="top" colspan="1" rowspan="1">
[
<a href="#ORdaniel-61" class="usa-link" aria-describedby="ORdaniel-61">61</a>
<a href="#ORdaniel-62" class="usa-link" aria-describedby="ORdaniel-62">62</a>
<a href="#ORdaniel-63" class="usa-link" aria-describedby="ORdaniel-63">63</a>
<a href="#ORdaniel-64" class="usa-link" aria-describedby="ORdaniel-64">64</a>
<a href="#ORdaniel-65" class="usa-link" aria-describedby="ORdaniel-65">65</a>
<a href="#ORdaniel-66" class="usa-link" aria-describedby="ORdaniel-66">66</a>
<a href="#ORdaniel-67" class="usa-link" aria-describedby="ORdaniel-67">67</a>
<a href="#ORdaniel-68" class="usa-link" aria-describedby="ORdaniel-68">68</a>
<a href="#ORdaniel-69" class="usa-link" aria-describedby="ORdaniel-69">69</a>
<a href="#ORdaniel-70" class="usa-link" aria-describedby="ORdaniel-70">70</a>
<a href="#ORdaniel-71" class="usa-link" aria-describedby="ORdaniel-71">71</a>
<a href="#ORdaniel-72" class="usa-link" aria-describedby="ORdaniel-72">72</a>
<a href="#ORdaniel-73" class="usa-link" aria-describedby="ORdaniel-73">73</a>
<a href="#ORdaniel-74" class="usa-link" aria-describedby="ORdaniel-74">74</a>
<a href="#ORdaniel-75" class="usa-link" aria-describedby="ORdaniel-75">75</a>
<a href="#ORdaniel-76" class="usa-link" aria-describedby="ORdaniel-76">76</a>
<a href="#ORdaniel-77" class="usa-link" aria-describedby="ORdaniel-77">77</a>
]
</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">Managing Study Data (focus semantic interoperability, data integration and standardization) (n=19)</td>
<td align="left" valign="top" colspan="1" rowspan="1">
[
<a href="#ORdaniel-78" class="usa-link" aria-describedby="ORdaniel-78">78</a>
<a href="#ORdaniel-79" class="usa-link" aria-describedby="ORdaniel-79">79</a>
<a href="#ORdaniel-80" class="usa-link" aria-describedby="ORdaniel-80">80</a>
<a href="#ORdaniel-81" class="usa-link" aria-describedby="ORdaniel-81">81</a>
<a href="#ORdaniel-82" class="usa-link" aria-describedby="ORdaniel-82">82</a>
<a href="#ORdaniel-83" class="usa-link" aria-describedby="ORdaniel-83">83</a>
<a href="#ORdaniel-84" class="usa-link" aria-describedby="ORdaniel-84">84</a>
<a href="#ORdaniel-85" class="usa-link" aria-describedby="ORdaniel-85">85</a>
<a href="#ORdaniel-86" class="usa-link" aria-describedby="ORdaniel-86">86</a>
<a href="#ORdaniel-87" class="usa-link" aria-describedby="ORdaniel-87">87</a>
<a href="#ORdaniel-88" class="usa-link" aria-describedby="ORdaniel-88">88</a>
<a href="#ORdaniel-89" class="usa-link" aria-describedby="ORdaniel-89">89</a>
<a href="#ORdaniel-90" class="usa-link" aria-describedby="ORdaniel-90">90</a>
<a href="#ORdaniel-91" class="usa-link" aria-describedby="ORdaniel-91">91</a>
<a href="#ORdaniel-92" class="usa-link" aria-describedby="ORdaniel-92">92</a>
<a href="#ORdaniel-93" class="usa-link" aria-describedby="ORdaniel-93">93</a>
<a href="#ORdaniel-94" class="usa-link" aria-describedby="ORdaniel-94">94</a>
<a href="#ORdaniel-95" class="usa-link" aria-describedby="ORdaniel-95">95</a>
<a href="#ORdaniel-96" class="usa-link" aria-describedby="ORdaniel-96">96</a>
]
</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">Managing Study Data (focus data quality assessment) (n=14)</td>
<td align="left" valign="top" colspan="1" rowspan="1">
[
<a href="#ORdaniel-97" class="usa-link" aria-describedby="ORdaniel-97">97</a>
<a href="#ORdaniel-98" class="usa-link" aria-describedby="ORdaniel-98">98</a>
<a href="#ORdaniel-99" class="usa-link" aria-describedby="ORdaniel-99">99</a>
<a href="#ORdaniel-100" class="usa-link" aria-describedby="ORdaniel-100">100</a>
<a href="#ORdaniel-101" class="usa-link" aria-describedby="ORdaniel-101">101</a>
<a href="#ORdaniel-102" class="usa-link" aria-describedby="ORdaniel-102">102</a>
<a href="#ORdaniel-103" class="usa-link" aria-describedby="ORdaniel-103">103</a>
<a href="#ORdaniel-104" class="usa-link" aria-describedby="ORdaniel-104">104</a>
<a href="#ORdaniel-105" class="usa-link" aria-describedby="ORdaniel-105">105</a>
<a href="#ORdaniel-106" class="usa-link" aria-describedby="ORdaniel-106">106</a>
<a href="#ORdaniel-107" class="usa-link" aria-describedby="ORdaniel-107">107</a>
<a href="#ORdaniel-108" class="usa-link" aria-describedby="ORdaniel-108">108</a>
<a href="#ORdaniel-109" class="usa-link" aria-describedby="ORdaniel-109">109</a>
<a href="#ORdaniel-110" class="usa-link" aria-describedby="ORdaniel-110">110</a>
]
</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">Managing Study Data (focus security, confidentiality) (n=9)</td>
<td align="left" valign="top" colspan="1" rowspan="1">
[
<a href="#ORdaniel-111" class="usa-link" aria-describedby="ORdaniel-111">111</a>
<a href="#ORdaniel-112" class="usa-link" aria-describedby="ORdaniel-112">112</a>
<a href="#ORdaniel-113" class="usa-link" aria-describedby="ORdaniel-113">113</a>
<a href="#ORdaniel-114" class="usa-link" aria-describedby="ORdaniel-114">114</a>
<a href="#ORdaniel-115" class="usa-link" aria-describedby="ORdaniel-115">115</a>
<a href="#ORdaniel-116" class="usa-link" aria-describedby="ORdaniel-116">116</a>
<a href="#ORdaniel-117" class="usa-link" aria-describedby="ORdaniel-117">117</a>
<a href="#ORdaniel-118" class="usa-link" aria-describedby="ORdaniel-118">118</a>
<a href="#ORdaniel-119" class="usa-link" aria-describedby="ORdaniel-119">119</a>
]
</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">MKD</td>
<td align="left" valign="top" colspan="1" rowspan="1">Data/text mining</td>
<td align="left" valign="top" colspan="1" rowspan="1">n=18</td>
<td align="left" valign="top" colspan="1" rowspan="1">Using Study Data (data or text mining, data visualization)</td>
<td align="left" valign="top" colspan="1" rowspan="1">CRI step 5: Using Study Data</td>
<td align="left" valign="top" colspan="1" rowspan="1">
[
<a href="#ORdaniel-120" class="usa-link" aria-describedby="ORdaniel-120">120</a>
<a href="#ORdaniel-121" class="usa-link" aria-describedby="ORdaniel-121">121</a>
<a href="#ORdaniel-122" class="usa-link" aria-describedby="ORdaniel-122">122</a>
<a href="#ORdaniel-123" class="usa-link" aria-describedby="ORdaniel-123">123</a>
<a href="#ORdaniel-124" class="usa-link" aria-describedby="ORdaniel-124">124</a>
<a href="#ORdaniel-125" class="usa-link" aria-describedby="ORdaniel-125">125</a>
<a href="#ORdaniel-126" class="usa-link" aria-describedby="ORdaniel-126">126</a>
<a href="#ORdaniel-127" class="usa-link" aria-describedby="ORdaniel-127">127</a>
<a href="#ORdaniel-128" class="usa-link" aria-describedby="ORdaniel-128">128</a>
<a href="#ORdaniel-129" class="usa-link" aria-describedby="ORdaniel-129">129</a>
<a href="#ORdaniel-130" class="usa-link" aria-describedby="ORdaniel-130">130</a>
<a href="#ORdaniel-131" class="usa-link" aria-describedby="ORdaniel-131">131</a>
<a href="#ORdaniel-132" class="usa-link" aria-describedby="ORdaniel-132">132</a>
<a href="#ORdaniel-133" class="usa-link" aria-describedby="ORdaniel-133">133</a>
<a href="#ORdaniel-134" class="usa-link" aria-describedby="ORdaniel-134">134</a>
<a href="#ORdaniel-135" class="usa-link" aria-describedby="ORdaniel-135">135</a>
<a href="#ORdaniel-136" class="usa-link" aria-describedby="ORdaniel-136">136</a>
<a href="#ORdaniel-137" class="usa-link" aria-describedby="ORdaniel-137">137</a>
]
</td>
</tr>
<tr>
<td align="left" rowspan="2" valign="top" colspan="1">COM-PUB</td>
<td align="left" rowspan="2" valign="top" colspan="1">Results communication and publication</td>
<td align="left" rowspan="2" valign="top" colspan="1">n=5</td>
<td align="left" valign="top" colspan="1" rowspan="1">Communicating Study Results (Research results dissemination)</td>
<td align="left" valign="top" colspan="1" rowspan="1">CRI step 6: Communicating Study Results</td>
<td align="left" valign="top" colspan="1" rowspan="1">
[
<a href="#ORdaniel-138" class="usa-link" aria-describedby="ORdaniel-138">138</a>
<a href="#ORdaniel-139" class="usa-link" aria-describedby="ORdaniel-139">139</a>
<a href="#ORdaniel-140" class="usa-link" aria-describedby="ORdaniel-140">140</a>
<a href="#ORdaniel-141" class="usa-link" aria-describedby="ORdaniel-141">141</a>
]
</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">Analyzing Study Publications (knowledge representation, management, or engineering)</td>
<td align="left" valign="top" colspan="1" rowspan="1">CRI step 7: Analyzing Study Publications</td>
<td align="left" valign="top" colspan="1" rowspan="1">
[
<a href="#ORdaniel-142" class="usa-link" aria-describedby="ORdaniel-142">142</a>
]
</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">INFRA</td>
<td align="left" valign="top" colspan="1" rowspan="1">Technical infrastructure</td>
<td align="left" valign="top" colspan="1" rowspan="1">n=7</td>
<td align="left" valign="top" colspan="1" rowspan="1">Architecture (Big Data, Cloud computing)</td>
<td align="left" valign="top" colspan="1" rowspan="1"></td>
<td align="left" valign="top" colspan="1" rowspan="1">
[
<a href="#ORdaniel-143" class="usa-link" aria-describedby="ORdaniel-143">143</a>
<a href="#ORdaniel-144" class="usa-link" aria-describedby="ORdaniel-144">144</a>
<a href="#ORdaniel-145" class="usa-link" aria-describedby="ORdaniel-145">145</a>
<a href="#ORdaniel-146" class="usa-link" aria-describedby="ORdaniel-146">146</a>
<a href="#ORdaniel-147" class="usa-link" aria-describedby="ORdaniel-147">147</a>
<a href="#ORdaniel-148" class="usa-link" aria-describedby="ORdaniel-148">148</a>
<a href="#ORdaniel-149" class="usa-link" aria-describedby="ORdaniel-149">149</a>
]
</td>
</tr>
<tr>
<td align="left" rowspan="2" valign="top" colspan="1">ETH-REG</td>
<td align="left" rowspan="2" valign="top" colspan="1">Ethical &amp; legal issues</td>
<td align="left" rowspan="2" valign="top" colspan="1">n=11</td>
<td align="left" valign="top" colspan="1" rowspan="1">Ethical issues</td>
<td align="left" valign="top" colspan="1" rowspan="1"></td>
<td align="left" rowspan="2" valign="top" colspan="1">
[
<a href="#ORdaniel-150" class="usa-link" aria-describedby="ORdaniel-150">150</a>
<a href="#ORdaniel-151" class="usa-link" aria-describedby="ORdaniel-151">151</a>
<a href="#ORdaniel-152" class="usa-link" aria-describedby="ORdaniel-152">152</a>
<a href="#ORdaniel-153" class="usa-link" aria-describedby="ORdaniel-153">153</a>
<a href="#ORdaniel-154" class="usa-link" aria-describedby="ORdaniel-154">154</a>
<a href="#ORdaniel-155" class="usa-link" aria-describedby="ORdaniel-155">155</a>
<a href="#ORdaniel-156" class="usa-link" aria-describedby="ORdaniel-156">156</a>
<a href="#ORdaniel-157" class="usa-link" aria-describedby="ORdaniel-157">157</a>
<a href="#ORdaniel-158" class="usa-link" aria-describedby="ORdaniel-158">158</a>
<a href="#ORdaniel-159" class="usa-link" aria-describedby="ORdaniel-159">159</a>
<a href="#ORdaniel-160" class="usa-link" aria-describedby="ORdaniel-160">160</a>
]
</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">Legal, regulatory issues</td>
<td align="left" valign="top" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="4" valign="top" colspan="1">SOC</td>
<td align="left" rowspan="4" valign="top" colspan="1">Societal issues</td>
<td align="left" rowspan="4" valign="top" colspan="1">n=27</td>
<td align="left" valign="top" colspan="1" rowspan="1">Societal issues (focus governance, stakeholder participation, engagement)</td>
<td align="left" valign="top" colspan="1" rowspan="1"></td>
<td align="left" rowspan="4" valign="top" colspan="1">
[
<a href="#ORdaniel-161" class="usa-link" aria-describedby="ORdaniel-161">161</a>
<a href="#ORdaniel-162" class="usa-link" aria-describedby="ORdaniel-162">162</a>
<a href="#ORdaniel-163" class="usa-link" aria-describedby="ORdaniel-163">163</a>
<a href="#ORdaniel-164" class="usa-link" aria-describedby="ORdaniel-164">164</a>
<a href="#ORdaniel-165" class="usa-link" aria-describedby="ORdaniel-165">165</a>
<a href="#ORdaniel-166" class="usa-link" aria-describedby="ORdaniel-166">166</a>
<a href="#ORdaniel-167" class="usa-link" aria-describedby="ORdaniel-167">167</a>
<a href="#ORdaniel-168" class="usa-link" aria-describedby="ORdaniel-168">168</a>
<a href="#ORdaniel-169" class="usa-link" aria-describedby="ORdaniel-169">169</a>
<a href="#ORdaniel-170" class="usa-link" aria-describedby="ORdaniel-170">170</a>
<a href="#ORdaniel-171" class="usa-link" aria-describedby="ORdaniel-171">171</a>
<a href="#ORdaniel-172" class="usa-link" aria-describedby="ORdaniel-172">172</a>
<a href="#ORdaniel-173" class="usa-link" aria-describedby="ORdaniel-173">173</a>
<a href="#ORdaniel-174" class="usa-link" aria-describedby="ORdaniel-174">174</a>
<a href="#ORdaniel-175" class="usa-link" aria-describedby="ORdaniel-175">175</a>
<a href="#ORdaniel-176" class="usa-link" aria-describedby="ORdaniel-176">176</a>
<a href="#ORdaniel-177" class="usa-link" aria-describedby="ORdaniel-177">177</a>
<a href="#ORdaniel-178" class="usa-link" aria-describedby="ORdaniel-178">178</a>
<a href="#ORdaniel-179" class="usa-link" aria-describedby="ORdaniel-179">179</a>
<a href="#ORdaniel-180" class="usa-link" aria-describedby="ORdaniel-180">180</a>
<a href="#ORdaniel-181" class="usa-link" aria-describedby="ORdaniel-181">181</a>
<a href="#ORdaniel-182" class="usa-link" aria-describedby="ORdaniel-182">182</a>
<a href="#ORdaniel-183" class="usa-link" aria-describedby="ORdaniel-183">183</a>
<a href="#ORdaniel-184" class="usa-link" aria-describedby="ORdaniel-184">184</a>
<a href="#ORdaniel-185" class="usa-link" aria-describedby="ORdaniel-185">185</a>
<a href="#ORdaniel-186" class="usa-link" aria-describedby="ORdaniel-186">186</a>
<a href="#ORdaniel-187" class="usa-link" aria-describedby="ORdaniel-187">187</a>
]
</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">Societal issues (focus patient participation, health literacy, patient consenting)</td>
<td align="left" valign="top" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">Societal issues (focus business model)</td>
<td align="left" valign="top" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">Societal issues (focus workforce, education and training)</td>
<td align="left" valign="top" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">LHS-SURV</td>
<td align="left" valign="top" colspan="1" rowspan="1">Learning Health Systems Surveillance</td>
<td align="left" valign="top" colspan="1" rowspan="1">n=10</td>
<td align="left" valign="top" colspan="1" rowspan="1">Learning Health Systems, disease, drug &amp; device surveillance</td>
<td align="left" valign="top" colspan="1" rowspan="1"></td>
<td align="left" valign="top" colspan="1" rowspan="1">
[
<a href="#ORdaniel-188" class="usa-link" aria-describedby="ORdaniel-188">188</a>
<a href="#ORdaniel-189" class="usa-link" aria-describedby="ORdaniel-189">189</a>
<a href="#ORdaniel-190" class="usa-link" aria-describedby="ORdaniel-190">190</a>
<a href="#ORdaniel-191" class="usa-link" aria-describedby="ORdaniel-191">191</a>
<a href="#ORdaniel-192" class="usa-link" aria-describedby="ORdaniel-192">192</a>
<a href="#ORdaniel-193" class="usa-link" aria-describedby="ORdaniel-193">193</a>
<a href="#ORdaniel-194" class="usa-link" aria-describedby="ORdaniel-194">194</a>
<a href="#ORdaniel-195" class="usa-link" aria-describedby="ORdaniel-195">195</a>
<a href="#ORdaniel-196" class="usa-link" aria-describedby="ORdaniel-196">196</a>
<a href="#ORdaniel-197" class="usa-link" aria-describedby="ORdaniel-197">197</a>
]
</td>
</tr>
<tr>
<td align="left" valign="top" colspan="1" rowspan="1">TRE</td>
<td align="left" valign="top" colspan="1" rowspan="1">Trends</td>
<td align="left" valign="top" colspan="1" rowspan="1">n=13</td>
<td align="left" valign="top" colspan="1" rowspan="1">Trends</td>
<td align="left" valign="top" colspan="1" rowspan="1"></td>
<td align="left" valign="top" colspan="1" rowspan="1">
[
<a href="#ORdaniel-1" class="usa-link" aria-describedby="ORdaniel-1">1</a>
<a href="#ORdaniel-2" class="usa-link" aria-describedby="ORdaniel-2">2</a>
<a href="#ORdaniel-3" class="usa-link" aria-describedby="ORdaniel-3">3</a>
<a href="#ORdaniel-4" class="usa-link" aria-describedby="ORdaniel-4">4</a>
<a href="#ORdaniel-5" class="usa-link" aria-describedby="ORdaniel-5">5</a>
,
<a href="#ORdaniel-198" class="usa-link" aria-describedby="ORdaniel-198">198</a>
<a href="#ORdaniel-199" class="usa-link" aria-describedby="ORdaniel-199">199</a>
<a href="#ORdaniel-200" class="usa-link" aria-describedby="ORdaniel-200">200</a>
<a href="#ORdaniel-201" class="usa-link" aria-describedby="ORdaniel-201">201</a>
<a href="#ORdaniel-202" class="usa-link" aria-describedby="ORdaniel-202">202</a>
<a href="#ORdaniel-203" class="usa-link" aria-describedby="ORdaniel-203">203</a>
<a href="#ORdaniel-204" class="usa-link" aria-describedby="ORdaniel-204">204</a>
<a href="#ORdaniel-205" class="usa-link" aria-describedby="ORdaniel-205">205</a>
]
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/TBdaniel-2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>
When the literature is plotted by categories over time (as in
<a href="#FIdaniel-3" class="usa-link">Figure 3</a>
), a different visualization of the evolution of the CRI field is highlighted, with different categories and topics varying by year.
</p>
<figure class="fig xbox font-sm" id="FIdaniel-3"><h5 class="obj_head">Figure 3.b.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/de91/12020646/361bdeb24295/10-1055-s-0044-1800732-idaniel-3.jpg" loading="lazy" height="228" width="800" alt="Figure 3.b"></p>
<div class="p text-right font-secondary"><a href="figure/FIdaniel-3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>
. Number of papers shortlisted through both AMIA and IMIA review processes (n=327) divided in the ten main categories over time.
. Number of selected/shortlisted papers in the four sub-categories of data management over time (n=90).
</p></figcaption></figure></section></section><section id="sec10"><h3 class="pmc_sec_title">3.2. Main CRI contributions supporting the life cycle of clinical research (2013-2022)</h3>
<p>Based on our combined analysis of CRI articles selected over time by the approaches described above, this section provides an overview from 10 years of publications demonstrating progress in the multifaceted aspects of CRI supporting research and innovation in healthcare and biomedicine. Each of the sections that follow cover major categories with exemplars highlighted within.</p>
<section id="sec11"><h4 class="pmc_sec_title">3.2.1. Study design, participant recruitment and study execution</h4>
<p>
To support the increasing complexities of clinical research over time, a number of initiatives, including the Human Studies Database [
<a href="#ORdaniel-15" class="usa-link" aria-describedby="ORdaniel-15">15</a>
], the database for aggregate analysis of ClinicalTrials.gov (AACT) [
<a href="#ORdaniel-7" class="usa-link" aria-describedby="ORdaniel-7">7</a>
], the more recent knowledge base of clinical trial eligibility criteria [
<a href="#ORdaniel-36" class="usa-link" aria-describedby="ORdaniel-36">36</a>
], and others [
<a href="#ORdaniel-21" class="usa-link" aria-describedby="ORdaniel-21">21</a>
,
<a href="#ORdaniel-35" class="usa-link" aria-describedby="ORdaniel-35">35</a>
] proposed access to clinical trials repositories allowing significant information to be leveraged and even re-used across studies. In addition, initiatives emerged that were aimed at optimizing clinical research deliverables while meeting the requirements enunciated by the U.S. Food and Drug Administration for electronic systems [
<a href="#ORdaniel-8" class="usa-link" aria-describedby="ORdaniel-8">8</a>
]. Major contributions over the last decade also focused on so-called
<em>secondary use</em>
of health data collected via routine care for optimizing clinical trials [
<a href="#ORdaniel-6" class="usa-link" aria-describedby="ORdaniel-6">6</a>
,
<a href="#ORdaniel-9" class="usa-link" aria-describedby="ORdaniel-9">9</a>
,
<a href="#ORdaniel-22" class="usa-link" aria-describedby="ORdaniel-22">22</a>
,
<a href="#ORdaniel-28" class="usa-link" aria-describedby="ORdaniel-28">28</a>
]. Indeed, many projects and initiatives have explored the value of EHR data for clinical trial design, patient recruitment [
<a href="#ORdaniel-12" class="usa-link" aria-describedby="ORdaniel-12">12</a>
<a href="#ORdaniel-13" class="usa-link" aria-describedby="ORdaniel-13">13</a>
<a href="#ORdaniel-14" class="usa-link" aria-describedby="ORdaniel-14">14</a>
,
<a href="#ORdaniel-17" class="usa-link" aria-describedby="ORdaniel-17">17</a>
,
<a href="#ORdaniel-19" class="usa-link" aria-describedby="ORdaniel-19">19</a>
,
<a href="#ORdaniel-25" class="usa-link" aria-describedby="ORdaniel-25">25</a>
,
<a href="#ORdaniel-26" class="usa-link" aria-describedby="ORdaniel-26">26</a>
,
<a href="#ORdaniel-30" class="usa-link" aria-describedby="ORdaniel-30">30</a>
,
<a href="#ORdaniel-31" class="usa-link" aria-describedby="ORdaniel-31">31</a>
,
<a href="#ORdaniel-33" class="usa-link" aria-describedby="ORdaniel-33">33</a>
,
<a href="#ORdaniel-34" class="usa-link" aria-describedby="ORdaniel-34">34</a>
,
<a href="#ORdaniel-37" class="usa-link" aria-describedby="ORdaniel-37">37</a>
,
<a href="#ORdaniel-43" class="usa-link" aria-describedby="ORdaniel-43">43</a>
,
<a href="#ORdaniel-180" class="usa-link" aria-describedby="ORdaniel-180">180</a>
], or for accelerating data collection [
<a href="#ORdaniel-11" class="usa-link" aria-describedby="ORdaniel-11">11</a>
,
<a href="#ORdaniel-39" class="usa-link" aria-describedby="ORdaniel-39">39</a>
<a href="#ORdaniel-40" class="usa-link" aria-describedby="ORdaniel-40">40</a>
<a href="#ORdaniel-41" class="usa-link" aria-describedby="ORdaniel-41">41</a>
<a href="#ORdaniel-42" class="usa-link" aria-describedby="ORdaniel-42">42</a>
]. Facilitated by the generalization of clinical data warehouses, some initiatives were carried out on a large scale within European projects such as the EHR4CR [
<a href="#ORdaniel-10" class="usa-link" aria-describedby="ORdaniel-10">10</a>
,
<a href="#ORdaniel-16" class="usa-link" aria-describedby="ORdaniel-16">16</a>
,
<a href="#ORdaniel-20" class="usa-link" aria-describedby="ORdaniel-20">20</a>
,
<a href="#ORdaniel-23" class="usa-link" aria-describedby="ORdaniel-23">23</a>
,
<a href="#ORdaniel-24" class="usa-link" aria-describedby="ORdaniel-24">24</a>
,
<a href="#ORdaniel-29" class="usa-link" aria-describedby="ORdaniel-29">29</a>
] or TRANSFoRm [
<a href="#ORdaniel-27" class="usa-link" aria-describedby="ORdaniel-27">27</a>
] projects. The EHR4CR project contributed to the development of the InSite commercial solution (formerly Custodix, Belgium, currently part of TriNetX) which reached the evaluation phase [
<a href="#ORdaniel-32" class="usa-link" aria-describedby="ORdaniel-32">32</a>
].
</p>
<p>
While randomized controlled trials (RCTs) are the gold standard for estimating treatment effects in medical research, there has also been growing interest in and use using real-world data (RWD) for use-cases including drug development, outcomes research, and observational studies. One such use case is the construction of external control arms for evaluation of efficacy in single-arm trials, particularly in cases where randomization is either infeasible or unethical [
<a href="#ORdaniel-44" class="usa-link" aria-describedby="ORdaniel-44">44</a>
,
<a href="#ORdaniel-46" class="usa-link" aria-describedby="ORdaniel-46">46</a>
,
<a href="#ORdaniel-47" class="usa-link" aria-describedby="ORdaniel-47">47</a>
]. In close alignment with a recently released draft framework by the Medical Device Innovation Consortium on real-world clinical evidence and
<em>in vitro</em>
diagnostics, publications report the possibility of substantially reducing the length and cost of diagnostic test evaluation for diseases with low prevalence to support regulatory decision-making by leveraging subjects from a RWD source [
<a href="#ORdaniel-38" class="usa-link" aria-describedby="ORdaniel-38">38</a>
]. The past decade has also seen the explosion of observational research using EHR data with CRI publications better illuminating our understanding of the specific caveats and considerations/guidelines for conducting observational studies and comparative effectiveness with RWD, while limiting biases [
<a href="#ORdaniel-18" class="usa-link" aria-describedby="ORdaniel-18">18</a>
].
</p></section><section id="sec12"><h4 class="pmc_sec_title">3.2.2. Data management</h4>
<section id="sec13"><h5 class="pmc_sec_title">Real-world data and Clinical Data warehouses</h5>
<p>
Clinical data warehouses (CDW), built early on at the level of healthcare facilities (Enterprise Data Warehouses) enabled evidence generation based on real-world data to become a reality during the last decade [
<a href="#ORdaniel-63" class="usa-link" aria-describedby="ORdaniel-63">63</a>
,
<a href="#ORdaniel-64" class="usa-link" aria-describedby="ORdaniel-64">64</a>
,
<a href="#ORdaniel-67" class="usa-link" aria-describedby="ORdaniel-67">67</a>
]. In the United States the CDW projects, supported by the Clinical and Translational Science Award (CTSA) funding program of the National Institutes of Health (NIH), have seen increasingly high levels of maturity [
<a href="#ORdaniel-62" class="usa-link" aria-describedby="ORdaniel-62">62</a>
,
<a href="#ORdaniel-75" class="usa-link" aria-describedby="ORdaniel-75">75</a>
]. One successful example is the Informatics for Integrating Biology and the Bedside (i2b2) solution that has been widely used by academic health centers in the U.S. and internationally. Notable i2b2 use-cases include discovery research, case-control studies [
<a href="#ORdaniel-66" class="usa-link" aria-describedby="ORdaniel-66">66</a>
], enable cost-effective genome-wide studies, identify important risks from commonly used medications [
<a href="#ORdaniel-61" class="usa-link" aria-describedby="ORdaniel-61">61</a>
], capture and evaluate patient reported outcomes [
<a href="#ORdaniel-48" class="usa-link" aria-describedby="ORdaniel-48">48</a>
], access image from clinical archive [
<a href="#ORdaniel-69" class="usa-link" aria-describedby="ORdaniel-69">69</a>
], and enable data sharing at scale [
<a href="#ORdaniel-70" class="usa-link" aria-describedby="ORdaniel-70">70</a>
,
<a href="#ORdaniel-79" class="usa-link" aria-describedby="ORdaniel-79">79</a>
,
<a href="#ORdaniel-92" class="usa-link" aria-describedby="ORdaniel-92">92</a>
]. Hospital-based CDWs and multi-site data-sharing activities have also emerged in Europe and Asia over the past decade [
<a href="#ORdaniel-65" class="usa-link" aria-describedby="ORdaniel-65">65</a>
,
<a href="#ORdaniel-71" class="usa-link" aria-describedby="ORdaniel-71">71</a>
,
<a href="#ORdaniel-72" class="usa-link" aria-describedby="ORdaniel-72">72</a>
,
<a href="#ORdaniel-76" class="usa-link" aria-describedby="ORdaniel-76">76</a>
].
</p></section><section id="sec14"><h5 class="pmc_sec_title">Interoperability and standards</h5>
<p>
Interoperability is another major pilar of CRI that has evolved. In the domain of clinical research, the Clinical Data Information Standards Committee (CDISC) has developed a number of standards for study design, data collection and analysis, and submission to the regulatory bodies [
<a href="#ORdaniel-91" class="usa-link" aria-describedby="ORdaniel-91">91</a>
]. The Operational Data Model (ODM) has been updated to enable the implementation of case report forms supporting more efficiently the complete clinical study lifecycle from the design phase to the collection of patient level data [
<a href="#ORdaniel-86" class="usa-link" aria-describedby="ORdaniel-86">86</a>
]. The complexity and heterogeneity of healthcare data collection and storage approaches has remained a motivating challenge, and there has been movement in the CRI domain toward certain common data standards.
</p>
<p>
Several research networks focused on cross-institutional data analyzes at scale developed so-called “common data models” (CDM) that require each participating system to transform their underlining data model to the relevant CDM. Examples include the OMOP CDM from the OHDSI consortium [
<a href="#ORdaniel-89" class="usa-link" aria-describedby="ORdaniel-89">89</a>
,
<a href="#ORdaniel-94" class="usa-link" aria-describedby="ORdaniel-94">94</a>
,
<a href="#ORdaniel-120" class="usa-link" aria-describedby="ORdaniel-120">120</a>
], the FDA Mini Sentinel's CDM, I2B2-SHRINE [
<a href="#ORdaniel-79" class="usa-link" aria-describedby="ORdaniel-79">79</a>
], STRIDE or SHARPn [
<a href="#ORdaniel-81" class="usa-link" aria-describedby="ORdaniel-81">81</a>
,
<a href="#ORdaniel-143" class="usa-link" aria-describedby="ORdaniel-143">143</a>
]. New experience has been gained from these projects and the resulting models have been analyzed and compared [
<a href="#ORdaniel-88" class="usa-link" aria-describedby="ORdaniel-88">88</a>
]. While some require transfer of data to a centralized solution, another approach involves federated data and metadata management. One example is the ISO/IEC 11179 standard, that has been demonstrated to semantically link disparate common data elements defined by different organizations [
<a href="#ORdaniel-80" class="usa-link" aria-describedby="ORdaniel-80">80</a>
]. Minimum data sets based on common data elements have been built for different disease areas [
<a href="#ORdaniel-56" class="usa-link" aria-describedby="ORdaniel-56">56</a>
,
<a href="#ORdaniel-83" class="usa-link" aria-describedby="ORdaniel-83">83</a>
,
<a href="#ORdaniel-93" class="usa-link" aria-describedby="ORdaniel-93">93</a>
].
</p>
<p>
Furthermore, some initiatives have been conducted to bridge standards used in clinical care with clinical research standards, often developed through parallel, and often disparate, efforts. Other harmonization efforts have been conducted within the SHARPn project in mapping Healthcare Clinical Element Models (CEMs) adopted by the Office of the National Coordinator (ONC) to data elements extracted from the CDISC templates [
<a href="#ORdaniel-85" class="usa-link" aria-describedby="ORdaniel-85">85</a>
]. More recently, relying on the increasing adoption of HL7's Fast Healthcare Interoperability Resources (FHIR) by the healthcare industry, many studies have reported the interest of FHIR integration profiles to facilitate the integration of care and research activities [
<a href="#ORdaniel-39" class="usa-link" aria-describedby="ORdaniel-39">39</a>
,
<a href="#ORdaniel-92" class="usa-link" aria-describedby="ORdaniel-92">92</a>
,
<a href="#ORdaniel-95" class="usa-link" aria-describedby="ORdaniel-95">95</a>
]. The HL7 Vulcan acceleration program
<a href="#FNdaniel-1" class="usa-link">1</a>
develops FHIR resources needed to execute prioritized use cases of secondary use of real-world data and especially EHR data. While existing information model standards (
<em>e.g.</em>
, ISO 13606, openEHR, HL7 FHIR, and CDISC) define the basic semantics of health data, semantic interoperability are enabled by medical terminologies, nomenclatures, and ontologies. It is the use of these terminologies through explicitly defined value sets (terminology binding) that brings unambiguous semantics to health data. Consequently, it remains challenging to implement the shift from the “syntactic” level to the “semantic” level of data exchange enabling smart services and applications supporting communication among clinicians, researchers, and healthcare providers. In this context, Semantic Web technologies provide solutions for managing and sharing knowledge and for making the health data FAIR (Findable, Accessible, Interoperable, and Reusable) [
<a href="#ORdaniel-87" class="usa-link" aria-describedby="ORdaniel-87">87</a>
]. In contrast to other major foundational models for clinical research informatics (
<em>e.g.</em>
, BRIDG, CDISC ODM/SDTM, HL7 FHIR, OMOP CDM), the Ontology of Clinical Research (OCRe) takes a logic-oriented ontological modeling approach and attempt to model, not only operational and administrative needs, but also study validity, confounding, and bias needed for assessing study design strength [
<a href="#ORdaniel-15" class="usa-link" aria-describedby="ORdaniel-15">15</a>
]. Beyond the design of ontologies in healthcare (
<em>e.g.</em>
, for epilepsy [
<a href="#ORdaniel-78" class="usa-link" aria-describedby="ORdaniel-78">78</a>
]), research in Semantic Web technologies also includes activities contributing to CRI such as developing efficient ontology editors, and integrating ontologies into healthcare information systems [
<a href="#ORdaniel-82" class="usa-link" aria-describedby="ORdaniel-82">82</a>
,
<a href="#ORdaniel-84" class="usa-link" aria-describedby="ORdaniel-84">84</a>
,
<a href="#ORdaniel-90" class="usa-link" aria-describedby="ORdaniel-90">90</a>
,
<a href="#ORdaniel-96" class="usa-link" aria-describedby="ORdaniel-96">96</a>
]
</p></section><section id="sec15"><h5 class="pmc_sec_title">Data quality assessment</h5>
<p>
Data quality (DQ) is essential to many CRI activities. Identifying the need for common, standardized approaches to DQ, Kahn
<em>et al.</em>
were motivated to initiate a community-based effort in collaboration with the Electronic Data Methods Forum (EDM) to harmonize terminologies in CRI [
<a href="#ORdaniel-99" class="usa-link" aria-describedby="ORdaniel-99">99</a>
]. This DQ framework has been extended beyond intrinsic aspects to cover technical, and contextual categories across the data life cycle enabling the assessment and management of RWD repositories to ensure fitness for purpose. An ontology for characterizing DQ for secondary use of EHR data has been proposed [
<a href="#ORdaniel-98" class="usa-link" aria-describedby="ORdaniel-98">98</a>
]. An increasing number of studies describe DQ assessment methods or tools measuring completeness of data items and other DQ dimensions [
<a href="#ORdaniel-100" class="usa-link" aria-describedby="ORdaniel-100">100</a>
,
<a href="#ORdaniel-107" class="usa-link" aria-describedby="ORdaniel-107">107</a>
,
<a href="#ORdaniel-109" class="usa-link" aria-describedby="ORdaniel-109">109</a>
,
<a href="#ORdaniel-110" class="usa-link" aria-describedby="ORdaniel-110">110</a>
]. Two recent reviews summarize health DQ issues and assessment practices [
<a href="#ORdaniel-104" class="usa-link" aria-describedby="ORdaniel-104">104</a>
,
<a href="#ORdaniel-108" class="usa-link" aria-describedby="ORdaniel-108">108</a>
]. Some initiatives involve patients to assess the quality of their clinical data [
<a href="#ORdaniel-103" class="usa-link" aria-describedby="ORdaniel-103">103</a>
]. Medical centers [
<a href="#ORdaniel-102" class="usa-link" aria-describedby="ORdaniel-102">102</a>
] or large-scale research platforms [
<a href="#ORdaniel-101" class="usa-link" aria-describedby="ORdaniel-101">101</a>
,
<a href="#ORdaniel-106" class="usa-link" aria-describedby="ORdaniel-106">106</a>
] are setting up continuous DQ evaluation and improvement programs.
</p>
<p>
With the increasing use of observational, non-experimental data for various purposes, the potential for introducing biases and confounding factors that are often hidden in the data must be addressed or at least carefully considered, such as when conducting RWD studies or developing and validating predictive models [
<a href="#ORdaniel-74" class="usa-link" aria-describedby="ORdaniel-74">74</a>
,
<a href="#ORdaniel-97" class="usa-link" aria-describedby="ORdaniel-97">97</a>
,
<a href="#ORdaniel-105" class="usa-link" aria-describedby="ORdaniel-105">105</a>
]. Recent retractions of articles published in high-profile journals reporting COVID-19 studies based on EHR demonstrate that conforming to best practices in developing robust research based on RWD is critical to promote and foster rigor, quality, and reliability of this rapidly growing field [
<a href="#ORdaniel-73" class="usa-link" aria-describedby="ORdaniel-73">73</a>
,
<a href="#ORdaniel-140" class="usa-link" aria-describedby="ORdaniel-140">140</a>
].
</p></section><section id="sec16"><h5 class="pmc_sec_title">Security and data privacy-enhancing techniques</h5>
<p>
New research opportunities as well as artificial intelligence and machine learning techniques raise the need for greater data access but also bring new potential risks to privacy [
<a href="#ORdaniel-113" class="usa-link" aria-describedby="ORdaniel-113">113</a>
]. The evaluation of this risk in releasing datasets is a major concern [
<a href="#ORdaniel-111" class="usa-link" aria-describedby="ORdaniel-111">111</a>
,
<a href="#ORdaniel-112" class="usa-link" aria-describedby="ORdaniel-112">112</a>
,
<a href="#ORdaniel-114" class="usa-link" aria-describedby="ORdaniel-114">114</a>
]. Many studies focus on de-identification methods and privacy-enhancing techniques [
<a href="#ORdaniel-111" class="usa-link" aria-describedby="ORdaniel-111">111</a>
,
<a href="#ORdaniel-115" class="usa-link" aria-describedby="ORdaniel-115">115</a>
,
<a href="#ORdaniel-119" class="usa-link" aria-describedby="ORdaniel-119">119</a>
], including homomorphic encryption techniques enabling federated analysis of RWD while complying with data protection requirements [
<a href="#ORdaniel-117" class="usa-link" aria-describedby="ORdaniel-117">117</a>
]. An increasing number of papers propose methods for generating synthetic health data [
<a href="#ORdaniel-116" class="usa-link" aria-describedby="ORdaniel-116">116</a>
,
<a href="#ORdaniel-118" class="usa-link" aria-describedby="ORdaniel-118">118</a>
].
</p></section><section id="sec17"><h5 class="pmc_sec_title">Large scale research platforms</h5>
<p>
Many informatics platforms enabling collaborative research using multi-institutional distributed heterogenous data continued to emerge during the last decade [
<a href="#ORdaniel-49" class="usa-link" aria-describedby="ORdaniel-49">49</a>
]. Large-scale data-sharing initiatives are developing at national level. As examples, in the U.S. the SHARPn project that has proposed a scalable and standards-driven infrastructure for secondary use of EHR data [
<a href="#ORdaniel-143" class="usa-link" aria-describedby="ORdaniel-143">143</a>
]. SHRINE implementations have been used for nationally scalable multi-site disease studies of autism co-morbidity, juvenile idiopathic arthritis, peripartum cardiomyopathy, colorectal cancer, diabetes, and others [
<a href="#ORdaniel-79" class="usa-link" aria-describedby="ORdaniel-79">79</a>
]. The PCORnet initiative links multiple sub-networks via adherence to a common data model. Taken together, PCORnet sites include clinical data from a cohort of 80 million patients and enable rapid access to patient populations for pragmatic clinical trials, epidemiological research, and patient-centered research on both common presentations and rare diseases [
<a href="#ORdaniel-53" class="usa-link" aria-describedby="ORdaniel-53">53</a>
,
<a href="#ORdaniel-58" class="usa-link" aria-describedby="ORdaniel-58">58</a>
].
</p>
<p>
In Europe, the Health Data Hub in France and the Medical Informatics Initiative in Germany are major national initiatives supporting secondary uses of health data [
<a href="#ORdaniel-72" class="usa-link" aria-describedby="ORdaniel-72">72</a>
,
<a href="#ORdaniel-77" class="usa-link" aria-describedby="ORdaniel-77">77</a>
]. In U.K., we can cite two major initiatives, the Clinical Practice Research Datalink (CPRD) [
<a href="#ORdaniel-51" class="usa-link" aria-describedby="ORdaniel-51">51</a>
] and the ClinicAl disease research using LInked Bespoke studies and Electronic health Records (CALIBER) programs enabling secondary use of nationwide big data from linked electronic health records to improve outcomes in respectively primary care and cardiovascular diseases [
<a href="#ORdaniel-57" class="usa-link" aria-describedby="ORdaniel-57">57</a>
].
</p>
<p>
At a broad international scale, the Observational Health Data Sciences and Informatics (OHDSI) initiative installed the OMOP common data model as a
<em>de facto</em>
standard for observational research based on large-scale real-word data and demonstrated the value of large-scale data sharing in the analysis of drug use [
<a href="#ORdaniel-89" class="usa-link" aria-describedby="ORdaniel-89">89</a>
]. Finally, during the Covid19 pandemic, national Covid databases were implemented (for example in US (e.g. NC3) [
<a href="#ORdaniel-60" class="usa-link" aria-describedby="ORdaniel-60">60</a>
], Germany or Spain [
<a href="#ORdaniel-93" class="usa-link" aria-describedby="ORdaniel-93">93</a>
]) and international data-sharing platforms were set up (such as 4CE [
<a href="#ORdaniel-149" class="usa-link" aria-describedby="ORdaniel-149">149</a>
] or SCOR [
<a href="#ORdaniel-59" class="usa-link" aria-describedby="ORdaniel-59">59</a>
] projects). In addition, CRI efforts resulted in practical implementations of biobank research systems [
<a href="#ORdaniel-50" class="usa-link" aria-describedby="ORdaniel-50">50</a>
,
<a href="#ORdaniel-54" class="usa-link" aria-describedby="ORdaniel-54">54</a>
].
</p></section></section><section id="sec18"><h4 class="pmc_sec_title">3.2.3. Data/text mining, artificial intelligence/machine learning, knowledge discovery</h4>
<p>
One of the most striking findings in CRI over the last decade is the exceptionally rapid development of highly flexible, reusable tools developed to explore, visualize and analyze complex data sets and enable training of artificial intelligence (AI) models that are likely to transform medicine. Growing access to both vast data sets and advanced computational capabilities are enabling the rise of AI technologies and methods for diagnostic or therapeutic decision-support, and other approaches of relevance to CRI. Many studies report the use of machine learning in pharmacovigilance or in precision medicine to predict outcomes such as hospital admissions or in-hospital mortality for specific populations. A significant research effort has emerged to enable federated learning of predictive models from federated data sets [
<a href="#ORdaniel-127" class="usa-link" aria-describedby="ORdaniel-127">127</a>
]. Particular attention is also paid to methodological and organizational issues related to studies using EHR data and to propose methods for bias reduction in studies using EHR data [
<a href="#ORdaniel-128" class="usa-link" aria-describedby="ORdaniel-128">128</a>
,
<a href="#ORdaniel-131" class="usa-link" aria-describedby="ORdaniel-131">131</a>
,
<a href="#ORdaniel-133" class="usa-link" aria-describedby="ORdaniel-133">133</a>
,
<a href="#ORdaniel-140" class="usa-link" aria-describedby="ORdaniel-140">140</a>
].
</p>
<p>
New infrastructures and methods for data integration/fusion, record linkage, data and text mining have enabled new data discovery opportunities. Significant research efforts in the field focused on the development and validation of phenotyping methods. Rule-based phenotype definitions are collaboratively developed and evaluated within initiatives such as eMERGE [
<a href="#ORdaniel-121" class="usa-link" aria-describedby="ORdaniel-121">121</a>
] or PheKB
<a href="#FNdaniel-2" class="usa-link">2</a>
[
<a href="#ORdaniel-122" class="usa-link" aria-describedby="ORdaniel-122">122</a>
] in U.S. or the CALIBER platform
<a href="#FNdaniel-3" class="usa-link">3</a>
[
<a href="#ORdaniel-57" class="usa-link" aria-describedby="ORdaniel-57">57</a>
], and the phenotype health data gateway
<a href="#FNdaniel-4" class="usa-link">4</a>
in U.K. Many studies exploit the advances in natural language processing (NLP) to extract clinical information from clinical texts [
<a href="#ORdaniel-120" class="usa-link" aria-describedby="ORdaniel-120">120</a>
,
<a href="#ORdaniel-126" class="usa-link" aria-describedby="ORdaniel-126">126</a>
,
<a href="#ORdaniel-135" class="usa-link" aria-describedby="ORdaniel-135">135</a>
]. Others have implemented and promoted innovative approaches for high-throughput phenotyping [
<a href="#ORdaniel-81" class="usa-link" aria-describedby="ORdaniel-81">81</a>
,
<a href="#ORdaniel-120" class="usa-link" aria-describedby="ORdaniel-120">120</a>
,
<a href="#ORdaniel-123" class="usa-link" aria-describedby="ORdaniel-123">123</a>
<a href="#ORdaniel-124" class="usa-link" aria-describedby="ORdaniel-124">124</a>
<a href="#ORdaniel-125" class="usa-link" aria-describedby="ORdaniel-125">125</a>
,
<a href="#ORdaniel-129" class="usa-link" aria-describedby="ORdaniel-129">129</a>
,
<a href="#ORdaniel-130" class="usa-link" aria-describedby="ORdaniel-130">130</a>
,
<a href="#ORdaniel-132" class="usa-link" aria-describedby="ORdaniel-132">132</a>
,
<a href="#ORdaniel-134" class="usa-link" aria-describedby="ORdaniel-134">134</a>
].
</p>
<section id="sec19"><h5 class="pmc_sec_title">Study results communication and publication</h5>
<p>
Although of undeniable interest in the generation of evidence, especially in areas where traditional clinical trials would be unethical or infeasible, RWD should be reused considering the highest research standards and updated guidelines such as the methodological reference for conducting observational studies developed by European Network of Centers for Pharmacoepidemiology and Pharmacovigilance, ENCePP () under the aegis of the EMA (European Medicines Agency) and the FDA (Food and Drug Administration) or by research communities [
<a href="#ORdaniel-140" class="usa-link" aria-describedby="ORdaniel-140">140</a>
,
<a href="#ORdaniel-141" class="usa-link" aria-describedby="ORdaniel-141">141</a>
]. For example, the CONSORT-AI (Consolidated Standards of Reporting Trials-Artificial Intelligence) extension and its companion statement for clinical trial protocols: SPIRIT-AI (Standard Protocol Items: Recommendations for Interventional Trials-Artificial Intelligence) from the EQUATOR network provide recommendations (checklist items) promoting transparency and completeness in reporting clinical trials for AI interventions [
<a href="#ORdaniel-205" class="usa-link" aria-describedby="ORdaniel-205">205</a>
]. These recommendations assist editors and peer reviewers, as well as the general readership, to understand, interpret and critically appraise the quality of clinical trial design and risk of bias in the reported outcomes. In the radiology domain, DECIDE-AI is a new reporting guideline for artificial intelligence studies in radiology. Some other specific concerns of study research communication are e.g., the increase of self-citation [
<a href="#ORdaniel-139" class="usa-link" aria-describedby="ORdaniel-139">139</a>
] or tracking and efficiently displaying research evidences [
<a href="#ORdaniel-138" class="usa-link" aria-describedby="ORdaniel-138">138</a>
,
<a href="#ORdaniel-142" class="usa-link" aria-describedby="ORdaniel-142">142</a>
].
</p></section><section id="sec20"><h5 class="pmc_sec_title">Technical infrastructure</h5>
<p>
In the current era of massive-scale digitalization of data and computationally-intensive quantitative data analytics, the CRI community has developed new infrastructure capacities that can process, analyze and store petabytes of health data [
<a href="#ORdaniel-145" class="usa-link" aria-describedby="ORdaniel-145">145</a>
,
<a href="#ORdaniel-146" class="usa-link" aria-describedby="ORdaniel-146">146</a>
,
<a href="#ORdaniel-148" class="usa-link" aria-describedby="ORdaniel-148">148</a>
]. The management of massive, unstructured and heterogeneous data is still challenging, especially for supporting the identification of environmental exposures as determinants of physio pathological processes and providing a coherent framework for dealing with multi-scale population data including the phenome, genome, exposome, and their interconnections in the context of the One Health paradigm [
<a href="#ORdaniel-65" class="usa-link" aria-describedby="ORdaniel-65">65</a>
,
<a href="#ORdaniel-144" class="usa-link" aria-describedby="ORdaniel-144">144</a>
,
<a href="#ORdaniel-147" class="usa-link" aria-describedby="ORdaniel-147">147</a>
].
</p></section><section id="sec21"><h5 class="pmc_sec_title">Ethical, legal and societal issues (data governance, patient engagement, sustainability)</h5>
<p>
A major lesson that the coronavirus disease 2019 (COVID-19) pandemic has taught the scientific CRI community is that healthcare institutions shall continue to strengthen their expertise in data driven evidence generation. Since new opportunities also bring new societal challenges, most of the institutions in charge of CRI and/or CDW for research put in place governance committees of varying forms in charge of setting up the policies of data requests for internal research projects and/or external data sharing agreements as well as reviewing these requests [
<a href="#ORdaniel-162" class="usa-link" aria-describedby="ORdaniel-162">162</a>
]. These governance bodies need to guarantee that the intensive data-driven research is conducted in respect of the law and in line with ethical principles (
<em>e.g.</em>
, that data release fits the purpose of the research), especially in the context of large-scale collaborative research. Ethical and regulatory issues related to the use of large-scale linked data and new AI technologies are especially challenging and need to be addressed to respond to the current rapidly changing data-driven agenda. Researchers have to understand complicated and sometimes contradictory legal requirements inherent to big data projects and to consider ethical obligations in order to balance potential of discovery with legal and ethical considerations [
<a href="#ORdaniel-151" class="usa-link" aria-describedby="ORdaniel-151">151</a>
,
<a href="#ORdaniel-152" class="usa-link" aria-describedby="ORdaniel-152">152</a>
,
<a href="#ORdaniel-154" class="usa-link" aria-describedby="ORdaniel-154">154</a>
,
<a href="#ORdaniel-159" class="usa-link" aria-describedby="ORdaniel-159">159</a>
,
<a href="#ORdaniel-160" class="usa-link" aria-describedby="ORdaniel-160">160</a>
]. Ethics review committees may have to be reformed to deal with big data research and improve their oversight capacity [
<a href="#ORdaniel-156" class="usa-link" aria-describedby="ORdaniel-156">156</a>
,
<a href="#ORdaniel-158" class="usa-link" aria-describedby="ORdaniel-158">158</a>
] and supportive tools (eIRB systems) could be useful in this context. Some publications focus on lessons learned and recommendations related to ethical, legal and societal issues in the specific context of the covid-19 pandemics [
<a href="#ORdaniel-157" class="usa-link" aria-describedby="ORdaniel-157">157</a>
] or rare diseases. In Europe, the European Institute for Innovation through Health Data (i~HD) plays a strong role in many national and international collaborative projects or initiatives involving academic research groups from all over Europe collaborating with global pharmaceutical companies in supporting the ecosystem in especially addressing ethical, legal, societal issues [
<a href="#ORdaniel-177" class="usa-link" aria-describedby="ORdaniel-177">177</a>
]. Key national initiatives are promoting CRI [
<a href="#ORdaniel-161" class="usa-link" aria-describedby="ORdaniel-161">161</a>
,
<a href="#ORdaniel-166" class="usa-link" aria-describedby="ORdaniel-166">166</a>
,
<a href="#ORdaniel-171" class="usa-link" aria-describedby="ORdaniel-171">171</a>
,
<a href="#ORdaniel-172" class="usa-link" aria-describedby="ORdaniel-172">172</a>
,
<a href="#ORdaniel-178" class="usa-link" aria-describedby="ORdaniel-178">178</a>
,
<a href="#ORdaniel-179" class="usa-link" aria-describedby="ORdaniel-179">179</a>
,
<a href="#ORdaniel-181" class="usa-link" aria-describedby="ORdaniel-181">181</a>
]. Significant research efforts in the CRI field also focused on specific societal issues : patient information and consent [
<a href="#ORdaniel-150" class="usa-link" aria-describedby="ORdaniel-150">150</a>
,
<a href="#ORdaniel-153" class="usa-link" aria-describedby="ORdaniel-153">153</a>
,
<a href="#ORdaniel-155" class="usa-link" aria-describedby="ORdaniel-155">155</a>
,
<a href="#ORdaniel-168" class="usa-link" aria-describedby="ORdaniel-168">168</a>
,
<a href="#ORdaniel-182" class="usa-link" aria-describedby="ORdaniel-182">182</a>
], patient involvement in the design and execution of data-intensive medical research and clinical trials [
<a href="#ORdaniel-164" class="usa-link" aria-describedby="ORdaniel-164">164</a>
,
<a href="#ORdaniel-170" class="usa-link" aria-describedby="ORdaniel-170">170</a>
,
<a href="#ORdaniel-175" class="usa-link" aria-describedby="ORdaniel-175">175</a>
,
<a href="#ORdaniel-176" class="usa-link" aria-describedby="ORdaniel-176">176</a>
,
<a href="#ORdaniel-183" class="usa-link" aria-describedby="ORdaniel-183">183</a>
], health equity in CRI [
<a href="#ORdaniel-180" class="usa-link" aria-describedby="ORdaniel-180">180</a>
,
<a href="#ORdaniel-184" class="usa-link" aria-describedby="ORdaniel-184">184</a>
,
<a href="#ORdaniel-186" class="usa-link" aria-describedby="ORdaniel-186">186</a>
], sustainability [
<a href="#ORdaniel-68" class="usa-link" aria-describedby="ORdaniel-68">68</a>
,
<a href="#ORdaniel-163" class="usa-link" aria-describedby="ORdaniel-163">163</a>
], the role of IT vendors [
<a href="#ORdaniel-167" class="usa-link" aria-describedby="ORdaniel-167">167</a>
,
<a href="#ORdaniel-185" class="usa-link" aria-describedby="ORdaniel-185">185</a>
], barriers to data sharing [
<a href="#ORdaniel-165" class="usa-link" aria-describedby="ORdaniel-165">165</a>
,
<a href="#ORdaniel-173" class="usa-link" aria-describedby="ORdaniel-173">173</a>
] and more generally educational issues [
<a href="#ORdaniel-169" class="usa-link" aria-describedby="ORdaniel-169">169</a>
,
<a href="#ORdaniel-174" class="usa-link" aria-describedby="ORdaniel-174">174</a>
].
</p></section><section id="sec22"><h5 class="pmc_sec_title">Learning Health System and large-scale surveillance</h5>
<p>
CRI contributions have also advanced support for the realization of a learning health system that enables healthcare systems to objectively and continuously monitor their practices, pragmatically compare the outcomes of care over time, including patient-reported outcomes, and simulate and evaluate the impacts of certain strategic decisions on organizations or the quality of care [
<a href="#ORdaniel-191" class="usa-link" aria-describedby="ORdaniel-191">191</a>
,
<a href="#ORdaniel-192" class="usa-link" aria-describedby="ORdaniel-192">192</a>
,
<a href="#ORdaniel-194" class="usa-link" aria-describedby="ORdaniel-194">194</a>
,
<a href="#ORdaniel-196" class="usa-link" aria-describedby="ORdaniel-196">196</a>
]. Major research efforts also focused on data-driven approaches for signal detection in pharmacovigilance especially in post marketing drug and vaccine safety surveillance and more generally patient safety [
<a href="#ORdaniel-188" class="usa-link" aria-describedby="ORdaniel-188">188</a>
<a href="#ORdaniel-189" class="usa-link" aria-describedby="ORdaniel-189">189</a>
<a href="#ORdaniel-190" class="usa-link" aria-describedby="ORdaniel-190">190</a>
,
<a href="#ORdaniel-193" class="usa-link" aria-describedby="ORdaniel-193">193</a>
,
<a href="#ORdaniel-195" class="usa-link" aria-describedby="ORdaniel-195">195</a>
].
</p></section></section></section><section id="sec23"><h3 class="pmc_sec_title">3.3. Recent (last two years) research trends and perspectives (2022-2023)</h3>
<section id="sec24"><h4 class="pmc_sec_title">3.3.1. Health technology assessment, maturity of CRI services and innovative clinical trial designs</h4>
<p>
Despite considerable progress over the past 10 plus years, certain challenges remain a focus for CRI now and into the future. CRI interventions still appear too often as anecdotes or quasi-experimental studies, whereas methodologically rigorous studies (
<em>e.g.</em>
, randomized, controlled studies) of CRI interventions remain rare. As the field matures, more efforts will be needed to demonstrate the successes and value of CRI interventions. Moreover, there is a need to enhance the level of formalization of CRI services in large academic centers.
<strong><em>Ensuring high-quality research</em></strong>
relies on multiple factors including variations in organizational culture of academic medical centers, retention of research workforces, efficient multi-departmental organizational collaborations (e.g., relationship between biomedical research expertise and enterprise IT), technical skills, operational efficiencies and transparent communication of standard policies, procedures and key performance indicators [
<a href="#ORdaniel-199" class="usa-link" aria-describedby="ORdaniel-199">199</a>
].
</p>
<p>
Another emerging concern relevant to CRI is the development, validation, and evaluation of
<strong><em>AI-based healthcare technology</em></strong>
. In the context of medical AI, that survived several “winters” since the 1970's, there is a need to better understand how new AI technologies will challenge current strategies for regulating and validating AI devices for medicine and research. Recent reviews focusing on the development of AI in medicine state that AI could transform clinician workflows, patient care, administrative tasks, and augment medical knowledge and decision making [
<a href="#ORdaniel-203" class="usa-link" aria-describedby="ORdaniel-203">203</a>
,
<a href="#ORdaniel-204" class="usa-link" aria-describedby="ORdaniel-204">204</a>
]. They also underline unsolved challenges, from issues about training ML systems to unclear accountability making AI's implementation difficult and largely ununderstood by healthcare professionals still struggling to implement AI in their daily practice. Guidelines for clinical trial protocols for interventions involving AI are available and some papers propose AI implementation frameworks and approaches for AI development, implementation, use, monitoring and governance [
<a href="#ORdaniel-131" class="usa-link" aria-describedby="ORdaniel-131">131</a>
,
<a href="#ORdaniel-133" class="usa-link" aria-describedby="ORdaniel-133">133</a>
,
<a href="#ORdaniel-198" class="usa-link" aria-describedby="ORdaniel-198">198</a>
].
</p>
<p>
As the regulatory environment becomes progressively receptive to utilizing real-world evidence in clinical trials, many publications focus on various
<strong><em>innovative hybrid clinical trial settings</em></strong>
. Of particular interest is the use of RWD to accelerate and perhaps even approximate certain traditional research approaches. Among innovative trial designs, platform trials including multiple trial arms (conducted simultaneously and/or sequentially) on different treatments under a single master protocol. The capabilities of medical centers to perform EHR-based protocol feasibility assessment, clinical site selection, and patient pre-screening are keys for successful platform trials [
<a href="#ORdaniel-45" class="usa-link" aria-describedby="ORdaniel-45">45</a>
]. An increasing number of initiatives, such as the TAES (TriAl Eligibility Surveillance) system, leverage existing real-world data, using artificial intelligence and standard data models, to enhance identification of patients potentially eligible for clinical trials and generate timely notifications to physicians and simultaneously decrease the burden on research teams of manual EHR review [
<a href="#ORdaniel-43" class="usa-link" aria-describedby="ORdaniel-43">43</a>
]. Target trial emulation can be used for eligibility determination [
<a href="#ORdaniel-200" class="usa-link" aria-describedby="ORdaniel-200">200</a>
] and also to assess agreement between clinical trials and analogue real-world evidence studies. Concordance in results vary depending on the agreement metric. Emulation differences, chance, and residual confounding can contribute to divergence in results [
<a href="#ORdaniel-201" class="usa-link" aria-describedby="ORdaniel-201">201</a>
].
</p></section><section id="sec25"><h4 class="pmc_sec_title">3.3.2. Translational research, better integration of care and research and Learning Health Systems</h4>
<p>
There is a need to promote organizational and technical interoperability between care and research domains towards the implementation of “
<strong><em>clinical trials as care option</em></strong>
”. There are an increasing number of initiatives aiming at avoiding double data entry through the integration of electronic case report forms (eCRF) systems with electronic health records (EHR) – the “
<strong><em>eSource” or “EHR2EDC” use case</em></strong>
. While technical capabilities are important, organizational priorities, structure, and the site's support of clinical research functions are equally important considerations. According to a recent study assessing site readiness for eSource, only 21% of sites were using Fast Healthcare Interoperability Resources (FHIR) standards to exchange patient data with other institutions. Respondents generally gave lower readiness for change ratings to organizations that did not have a separate research information technology group and where researchers practiced in hospitals not operated by their medical schools [
<a href="#ORdaniel-41" class="usa-link" aria-describedby="ORdaniel-41">41</a>
]. Although a recent study demonstrated that secure, consistent, and automated end-to-end data transmission from the treating physician to the regulatory authority was feasible, the industry-wide implementation of EHR2EDC still requires policy decisions that set the framework for the use of research data based on routine care data [
<a href="#ORdaniel-42" class="usa-link" aria-describedby="ORdaniel-42">42</a>
]. To overcome the difficulty of integration with EHR systems due to information security and data protection restrictions, a novel architecture for a federated electronic data capture system (fEDC) has been proposed [
<a href="#ORdaniel-202" class="usa-link" aria-describedby="ORdaniel-202">202</a>
].
</p>
<p>
Despite recent efforts, especially those led by HL7 organization, for example the
<strong><em>HL7 FHIR Vulcan accelerator</em></strong>
, formal representation of multimodal and multi-level data supporting data interoperability across clinical research and care domains is still challenging. Additional efforts are required from standard development organizations (CDISC, HL7 and DICOM) to better align the standards used in care and research domain and from the vendors of clinical and biomedical research applications to implement interoperability by design in the development of their solutions within a “single source” approach. Common data models solve many challenges of standardizing electronic health records (EHR) data but are unable to semantically integrate all of the resources [
<a href="#ORdaniel-96" class="usa-link" aria-describedby="ORdaniel-96">96</a>
]. Moreover, semantic interoperability enabling secondary use of clinical data requires a rigorous governance process to ensure internationally the quality of the data standardization process consistently across care and research domains. Recognizing that interoperability frameworks must be continuously adapted to the user's needs and that the update can hardly be fully automatized, a clear collaborative process is needed to efficiently support the creation of new semantic resources scoped to any additional use case of biomedical research.
</p>
<p>
The persistent challenges posed by data collection based on paradigms of
<strong><em>primary use of data vs. secondary use</em></strong>
remain a threat to achieving high quality/value data at the source. Currently considerable resources are dedicated to secondary use of health data and to complex pipelines curating and enriching data outside of the scope of patient care. Efforts to promote improved data quality by design in clinical settings will result in higher quality data for research and evidence generation. For ethical reasons and for supporting the implementation of the
<strong><em>Learning Heath System,</em></strong>
ensuring the quality as well as the privacy-preserving accessibility of clinical data for primary use should be considered a priority with major impacts on secondary use and the research agenda.
</p></section><section id="sec26"><h4 class="pmc_sec_title">3.3.3. Trends related to the technological pillar</h4>
<p>
Progress is being made in
<strong><em>data quality management</em></strong>
systems standardizing medical data extraction and quality control methods [
<a href="#ORdaniel-108" class="usa-link" aria-describedby="ORdaniel-108">108</a>
<a href="#ORdaniel-109" class="usa-link" aria-describedby="ORdaniel-109">109</a>
<a href="#ORdaniel-110" class="usa-link" aria-describedby="ORdaniel-110">110</a>
]. The development of methods and tools to promote the use of
<strong><em>textual data and advanced NLP methods</em></strong>
in real-world observational studies is still an active research area with important contributions from such groups as the NLP Working Group at the Observational Health Data Sciences and Informatics (OHDSI) consortium[
<a href="#ORdaniel-135" class="usa-link" aria-describedby="ORdaniel-135">135</a>
]. Major research efforts are also devoted to
<strong><em>phenotyping</em></strong>
. Clinical experts and computer scientists still experience a variety of challenges when building phenotypes, including challenges to discerning key clinical events, clinical reasoning, and temporal elements [
<a href="#ORdaniel-136" class="usa-link" aria-describedby="ORdaniel-136">136</a>
]. Standard representations for phenotypes are increasingly being promoted to support the creation, localization and sharing of highly specific phenotype algorithms [
<a href="#ORdaniel-96" class="usa-link" aria-describedby="ORdaniel-96">96</a>
,
<a href="#ORdaniel-137" class="usa-link" aria-describedby="ORdaniel-137">137</a>
]. Additional research is also needed for building secure and efficient infrastructures for “Big data” management supporting, for instance, federated machine learning on distributed data sets hosted on sovereign cloud environments [
<a href="#ORdaniel-127" class="usa-link" aria-describedby="ORdaniel-127">127</a>
].
</p></section><section id="sec27"><h4 class="pmc_sec_title">3.3.4. Trends related to the ethical, legal and societal issues (ELSI)</h4>
<p>
Ethical, legal and societal research continues to be needed to define and ensure an ethically-guided, fair and patient-centric approach for CRI. Such work is essential to better understand and solve practical issues related to potential biases and inequities arising from the use of RWD and AI [
<a href="#ORdaniel-186" class="usa-link" aria-describedby="ORdaniel-186">186</a>
,
<a href="#ORdaniel-187" class="usa-link" aria-describedby="ORdaniel-187">187</a>
]. Specifically, strategies for appropriate constituent engagement are essential to ensure that solutions like AI can meaningfully benefit patients and other end users. As progress continues toward a new era of digital health driven by cloud data storage, distributed computing, and machine learning, key stakeholders must collaborate to ensure research findings and their downstream implementations are inclusive, representative and accessible for all who should benefit, and work in CRI is key to this.
</p></section></section></section><section id="sec28"><h2 class="pmc_sec_title">4. Conclusions</h2>
<p>As stated above, our goal was to provide an updated practical and informative overview of the CRI field via a scoping reviewing of CRI publications over the last decade. By leveraging two annual review processes, the “AMIA CRI year-in-review” and IMIA yearbook approach, we started with over 1,500 publications, and ultimately selected 205 impactful papers to analyze in-depth for the current review.</p>
<p>The review results provide an overview of (a) the evolving scope of CRI activities based on, (b) a decade of research in CRI, highlighting major evaluations in core CRI discoveries, innovations, and practical CRI methods and tools, and (c) more recent (past 2 years) trends and areas of CRI emphasis. The findings, as well as examples of enabled large scale research projects across institutions, regionally, nationally and internationally, demonstrate a continued maturation of the field. The findings also show that some important topics remain in focus, reflecting ongoing challenges in CRI that have not yet been fully addressed. They also reveal a need for improved and more rigorous evaluation of CRI solutions and socio-technical interventions. More research is also needed to better integrate the care and research workflows and to implement and realize the goals of a Learning Heath Systems. Similarly, increased focus on building and evaluating robust technical, governance, ethical and policy approaches is essential to build and operate the secure, effective, equitable and efficient infrastructures needed for improving clinical and translational research in an increasingly data-rich, interconnected, and computationally powerful environments. As the field continues to mature, the trajectory of progress and impact that CRI has had over the past decade can be expected to continue, and that should accrue benefits for all.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Both authors contributed to the conception, drafting and major editing of the manuscript. The authors have no relevant disclosures to report.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="FNdaniel-1">
<sup>1</sup><p class="display-inline">
<a href="http://hl7.org/fhir/uv/vulcan-rwd/#overview" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://hl7.org/fhir/uv/vulcan-rwd/#overview</a>
</p>
</div>
<div class="fn p" id="FNdaniel-2">
<sup>2</sup><p class="display-inline">
<a href="http://phekb.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://phekb.org</a>
</p>
</div>
<div class="fn p" id="FNdaniel-3">
<sup>3</sup><p class="display-inline">
<a href="https://www.caliberresearch.org/portal" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.caliberresearch.org/portal</a>
</p>
</div>
<div class="fn p" id="FNdaniel-4">
<sup>4</sup><p class="display-inline">
<a href="https://phenotypes.healthdatagateway.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phenotypes.healthdatagateway.org/</a>
</p>
</div>
</div></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="ORdaniel-1">
<span class="label">1.</span><cite>Embi PJ, Payne PRO. Clinical research informatics: challenges, opportunities and definition for an emerging domain. J Am Med Inform Assoc JAMIA 2009;16:316–27. https://doi.org/10.1197/jamia.M3005.</cite> [<a href="https://doi.org/10.1197/jamia.M3005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2732242/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19261934/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-2">
<span class="label">2.</span><cite>Weng C, Kahn MG. Clinical Research Informatics for Big Data and Precision Medicine. Yearb Med Inform 2016:211–8. https://doi.org/10.15265/IY-2016-019.</cite> [<a href="https://doi.org/10.15265/IY-2016-019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5171548/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27830253/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-3">
<span class="label">3.</span><cite>Johnson SB. Clinical Research Informatics: Supporting the Research Study Lifecycle. Yearb Med Inform 2017;26:193–200. https://doi.org/10.15265/IY-2017-022.</cite> [<a href="https://doi.org/10.15265/IY-2017-022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6239240/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29063565/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-4">
<span class="label">4.</span><cite>Embi PJ. Clinical research informatics: survey of recent advances and trends in a maturing field. Yearb Med Inform 2013;8:178–84.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/23974569/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-5">
<span class="label">5.</span><cite>Solomonides A. Review of Clinical Research Informatics. Yearb Med Inform 2020;29:193–202. https://doi.rg/10.1055/s-0040-1701988.</cite> [<a href="https://doi.org/10.1055/s-0040-1701988" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7442526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32823316/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-6">
<span class="label">6.</span><cite>
Murphy SN, Dubey A, Embi PJ, Harris PA, Richter BG, Turisco F,
<em>et al.</em>
Current state of information technologies for the clinical research enterprise across academic medical centers. Clin Transl Sci 2012;5:281–4. https://doi.org/10.1111/j.1752-8062.2011.00387.x.
</cite> [<a href="https://doi.org/10.1111/j.1752-8062.2011.00387.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5439868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22686207/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-7">
<span class="label">7.</span><cite>
Tasneem A, Aberle L, Ananth H, Chakraborty S, Chiswell K, McCourt BJ,
<em>et al.</em>
The database for aggregate analysis of ClinicalTrials.gov (AACT) and subsequent regrouping by clinical specialty. PloS One 2012;7:e33677. https://doi.org/10.1371/journal.pone.0033677.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0033677" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3306288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22438982/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-8">
<span class="label">8.</span><cite>Bansal A, Chamberlain R, Karr S, Kwasa S, McLaughlin B, Nguyen B, et al. A 21 CFR Part 11 compliant graphically based electronic system for clinical research documentation. J Med Syst 2012;36:1661–72. https://doi.org/10.1007/s10916-010-9627-4.</cite> [<a href="https://doi.org/10.1007/s10916-010-9627-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21107888/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-9">
<span class="label">9.</span><cite>Marsolo K. Informatics and operations-let's get integrated. J Am Med Inform Assoc JAMIA 2013;20:122–4. https://doi.org/10.1136/amiajnl-2012-001194.</cite> [<a href="https://doi.org/10.1136/amiajnl-2012-001194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3555339/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22940670/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-10">
<span class="label">10.</span><cite>Coorevits P, Sundgren M, Klein GO, Bahr A, Claerhout B, Daniel C, et al. Electronic health records: new opportunities for clinical research. J Intern Med 2013;274:547–60. https://doi.org/10.1111/joim.12119.</cite> [<a href="https://doi.org/10.1111/joim.12119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23952476/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-11">
<span class="label">11.</span><cite>
Köpcke F, Kraus S, Scholler A, Nau C, Schüttler J, Prokosch H-U,
<em>et al.</em>
Secondary use of routinely collected patient data in a clinical trial: an evaluation of the effects on patient recruitment and data acquisition. Int J Med Inf 2013;82:185–92. https://doi.org/10.1016/j.ijmedinf.2012.11.008.
</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2012.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23266063/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-12">
<span class="label">12.</span><cite>
Weng C, Li Y, Berhe S, Boland MR, Gao J, Hruby GW,
<em>et al.</em>
An Integrated Model for Patient Care and Clinical Trials (IMPACT) to support clinical research visit scheduling workflow for future learning health systems. J Biomed Inform 2013;46:642–52. https://doi.org/10.1016/j.jbi.2013.05.001.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2013.05.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3716847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23684593/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-13">
<span class="label">13.</span><cite>
Shivade C, Raghavan P, Fosler-Lussier E, Embi PJ, Elhadad N, Johnson SB,
<em>et al.</em>
A review of approaches to identifying patient phenotype cohorts using electronic health records. J Am Med Inform Assoc JAMIA 2014;21:221–30. https://doi.org/10.1136/amiajnl-2013-001935.
</cite> [<a href="https://doi.org/10.1136/amiajnl-2013-001935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3932460/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24201027/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-14">
<span class="label">14.</span><cite>
Fernández-Breis JT, Maldonado JA, Marcos M, Legaz-García M del C, Moner D, Torres-Sospedra J,
<em>et al.</em>
Leveraging electronic healthcare record standards and semantic web technologies for the identification of patient cohorts. J Am Med Inform Assoc JAMIA 2013;20:e288-296. https://doi.org/10.1136/amiajnl-2013-001923.
</cite> [<a href="https://doi.org/10.1136/amiajnl-2013-001923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3861938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23934950/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-15">
<span class="label">15.</span><cite>
Sim I, Tu SW, Carini S, Lehmann HP, Pollock BH, Peleg M,
<em>et al.</em>
The Ontology of Clinical Research (OCRe): an informatics foundation for the science of clinical research. J Biomed Inform 2014;52:78–91. https://doi.org/10.1016/j.jbi.2013.11.002.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2013.11.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4019723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24239612/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-16">
<span class="label">16.</span><cite>Doods J, Botteri F, Dugas M, Fritz F, EHR4CR WP7. A European inventory of common electronic health record data elements for clinical trial feasibility. Trials 2014;15:18. https://doi.org/10.1186/1745-6215-15-18.</cite> [<a href="https://doi.org/10.1186/1745-6215-15-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3895709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24410735/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-17">
<span class="label">17.</span><cite>
Schreiweis B, Trinczek B, Köpcke F, Leusch T, Majeed RW, Wenk J,
<em>et al.</em>
Comparison of electronic health record system functionalities to support the patient recruitment process in clinical trials. Int J Med Inf 2014;83:860–8. https://doi.org/10.1016/j.ijmedinf.2014.08.005.
</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2014.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25189709/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-18">
<span class="label">18.</span><cite>Richesson RL, Horvath MM, Rusincovitch SA. Clinical research informatics and electronic health record data. Yearb Med Inform 2014;9:215–23. https://doi.org/10.15265/IY-2014-0009.</cite> [<a href="https://doi.org/10.15265/IY-2014-0009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4287078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25123746/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-19">
<span class="label">19.</span><cite>Miotto R, Weng C. Case-based reasoning using electronic health records efficiently identifies eligible patients for clinical trials. J Am Med Inform Assoc JAMIA 2015;22:e141-150. https://doi.org/10.1093/jamia/ocu050.</cite> [<a href="https://doi.org/10.1093/jamia/ocu050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4428438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25769682/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-20">
<span class="label">20.</span><cite>
Soto-Rey I, Trinczek B, Girardeau Y, Zapletal E, Ammour N, Doods J,
<em>et al.</em>
Efficiency and effectiveness evaluation of an automated multi-country patient count cohort system. BMC Med Res Methodol 2015;15:44. https://doi.org/10.1186/s12874-015-0035-9.
</cite> [<a href="https://doi.org/10.1186/s12874-015-0035-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4423123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25928269/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-21">
<span class="label">21.</span><cite>
Wu DTY, Hanauer DA, Mei Q, Clark PM, An LC, Proulx J,
<em>et al.</em>
Assessing the readability of ClinicalTrials.gov. J Am Med Inform Assoc JAMIA 2016;23:269–75. https://doi.org/10.1093/jamia/ocv062.
</cite> [<a href="https://doi.org/10.1093/jamia/ocv062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5009924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26269536/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-22">
<span class="label">22.</span><cite>Dugas M. Clinical Research Informatics: Recent Advances and Future Directions. Yearb Med Inform 2015;10:174–7. https://doi.org/10.15265/IY-2015-010.</cite> [<a href="https://doi.org/10.15265/IY-2015-010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4587057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26293865/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-23">
<span class="label">23.</span><cite>
De Moor G, Sundgren M, Kalra D, Schmidt A, Dugas M, Claerhout B,
<em>et al.</em>
Using electronic health records for clinical research: the case of the EHR4CR project. J Biomed Inform 2015;53:162–73. https://doi.org/10.1016/j.jbi.2014.10.006.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2014.10.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25463966/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-24">
<span class="label">24.</span><cite>
Beresniak A, Schmidt A, Proeve J, Bolanos E, Patel N, Ammour N,
<em>et al.</em>
Cost-benefit assessment of using electronic health records data for clinical research versus current practices: Contribution of the Electronic Health Records for Clinical Research (EHR4CR) European Project. Contemp Clin Trials 2016;46:85–91. https://doi.org/10.1016/j.cct.2015.11.011.
</cite> [<a href="https://doi.org/10.1016/j.cct.2015.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26600286/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-25">
<span class="label">25.</span><cite>Ateya MB, Delaney BC, Speedie SM. The value of structured data elements from electronic health records for identifying subjects for primary care clinical trials. BMC Med Inform Decis Mak 2016;16:1. https://doi.org/10.1186/s12911-016-0239-x.</cite> [<a href="https://doi.org/10.1186/s12911-016-0239-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4709934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26754574/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-26">
<span class="label">26.</span><cite>Eubank MH, Hyman DM, Kanakamedala AD, Gardos SM, Wills JM, Stetson PD. Automated eligibility screening and monitoring for genotype-driven precision oncology trials. J Am Med Inform Assoc JAMIA 2016;23:777–81. https://doi.org/10.1093/jamia/ocw020.</cite> [<a href="https://doi.org/10.1093/jamia/ocw020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6370254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27016727/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-27">
<span class="label">27.</span><cite>
Mastellos N, Bliźniuk G, Czopnik D, McGilchrist M, Misiaszek A, Bródka P,
<em>et al.</em>
Feasibility and acceptability of TRANSFoRm to improve clinical trial recruitment in primary care. Fam Pract 2016;33:186–91. https://doi.org/10.1093/fampra/cmv102.
</cite> [<a href="https://doi.org/10.1093/fampra/cmv102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26711958/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-28">
<span class="label">28.</span><cite>Hruby GW, Hoxha J, Ravichandran PC, Mendonça EA, Hanauer DA, Weng C. A data-driven concept schema for defining clinical research data needs. Int J Med Inf 2016;91:1–9. https://doi.org/10.1016/j.ijmedinf.2016.03.008.</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2016.03.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4873775/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27185504/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-29">
<span class="label">29.</span><cite>
Girardeau Y, Doods J, Zapletal E, Chatellier G, Daniel C, Burgun A,
<em>et al.</em>
Leveraging the EHR4CR platform to support patient inclusion in academic studies: challenges and lessons learned. BMC Med Res Methodol 2017;17:36. https://doi.org/10.1186/s12874-017-0299-3.
</cite> [<a href="https://doi.org/10.1186/s12874-017-0299-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5329914/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28241798/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-30">
<span class="label">30.</span><cite>
Krischer J, Cronholm PF, Burroughs C, McAlear CA, Borchin R, Easley E,
<em>et al.</em>
Experience With Direct-to-Patient Recruitment for Enrollment Into a Clinical Trial in a Rare Disease: A Web-Based Study. J Med Internet Res 2017;19:e50. https://doi.org/10.2196/jmir.6798.
</cite> [<a href="https://doi.org/10.2196/jmir.6798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5350442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28246067/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-31">
<span class="label">31.</span><cite>
Obeid JS, Beskow LM, Rape M, Gouripeddi R, Black RA, Cimino JJ,
<em>et al.</em>
A survey of practices for the use of electronic health records to support research recruitment. J Clin Transl Sci 2017;1:246–52. https://doi.org/10.1017/cts.2017.301.
</cite> [<a href="https://doi.org/10.1017/cts.2017.301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5890320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29657859/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-32">
<span class="label">32.</span><cite>
Claerhout B, Kalra D, Mueller C, Singh G, Ammour N, Meloni L,
<em>et al.</em>
Federated electronic health records research technology to support clinical trial protocol optimization: Evidence from EHR4CR and the InSite platform. J Biomed Inform 2019;90:103090. https://doi.org/10.1016/j.jbi.2018.12.004.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2018.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30611012/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-33">
<span class="label">33.</span><cite>Meystre SM, Heider PM, Kim Y, Aruch DB, Britten CD. Automatic trial eligibility surveillance based on unstructured clinical data. Int J Med Inf 2019;129:13–9. https://doi.org/10.1016/j.ijmedinf.2019.05.018.</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2019.05.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6717538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31445247/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-34">
<span class="label">34.</span><cite>Miller HN, Gleason KT, Juraschek SP, Plante TB, Lewis-Land C, Woods B, et al. Electronic medical record-based cohort selection and direct-to-patient, targeted recruitment: early efficacy and lessons learned. J Am Med Inform Assoc JAMIA 2019;26:1209–17. https://doi.org/10.1093/jamia/ocz168.</cite> [<a href="https://doi.org/10.1093/jamia/ocz168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6798572/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31553434/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-35">
<span class="label">35.</span><cite>
Kim JH, Ta CN, Liu C, Sung C, Butler AM, Stewart LA,
<em>et al.</em>
Towards clinical data-driven eligibility criteria optimization for interventional COVID-19 clinical trials. J Am Med Inform Assoc JAMIA 2021;28:14–22. https://doi.org/10.1093/jamia/ocaa276.
</cite> [<a href="https://doi.org/10.1093/jamia/ocaa276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7798960/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33260201/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-36">
<span class="label">36.</span><cite>Liu H, Chi Y, Butler A, Sun Y, Weng C. A knowledge base of clinical trial eligibility criteria. J Biomed Inform 2021;117:103771. https://doi.org/10.1016/j.jbi.2021.103771.</cite> [<a href="https://doi.org/10.1016/j.jbi.2021.103771" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8407851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33813032/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-37">
<span class="label">37.</span><cite>Rogers JR, Hripcsak G, Cheung YK, Weng C. Clinical comparison between trial participants and potentially eligible patients using electronic health record data: A generalizability assessment method. J Biomed Inform 2021;119:103822. https://doi.org/10.1016/j.jbi.2021.103822.</cite> [<a href="https://doi.org/10.1016/j.jbi.2021.103822" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8260457/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34044156/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-38">
<span class="label">38.</span><cite>
Chen W-C, Li H, Wang C, Lu N, Song C, Tiwari R,
<em>et al.</em>
Evaluation of diagnostic tests for low prevalence diseases: a statistical approach for leveraging real-world data to accelerate the study. J Biopharm Stat 2021;31:375–90. https://doi.org/10.1080/10543406.2021.1877724.
</cite> [<a href="https://doi.org/10.1080/10543406.2021.1877724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33615997/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-39">
<span class="label">39.</span><cite>
Cheng AC, Duda SN, Taylor R, Delacqua F, Lewis AA, Bosler T,
<em>et al.</em>
REDCap on FHIR: Clinical Data Interoperability Services. J Biomed Inform 2021;121:103871. https://doi.org/10.1016/j.jbi.2021.103871.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2021.103871" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9217161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34298155/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-40">
<span class="label">40.</span><cite>
Cheng AC, Banasiewicz MK, Johnson JD, Sulieman L, Kennedy N, Delacqua F,
<em>et al.</em>
Evaluating automated electronic case report form data entry from electronic health records. J Clin Transl Sci 2023;7:e29. https://doi.org/10.1017/cts.2022.514.
</cite> [<a href="https://doi.org/10.1017/cts.2022.514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9947602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36845316/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-41">
<span class="label">41.</span><cite>
Eisenstein EL, Zozus MN, Garza MY, Lanham HJ, Adagarla B, Walden A,
<em>et al.</em>
Assessing clinical site readiness for electronic health record (EHR)-to-electronic data capture (EDC) automated data collection. Contemp Clin Trials 2023;128:107144. https://doi.org/10.1016/j.cct.2023.107144.
</cite> [<a href="https://doi.org/10.1016/j.cct.2023.107144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10164080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36898625/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-42">
<span class="label">42.</span><cite>
Mueller C, Herrmann P, Cichos S, Remes B, Junker E, Hastenteufel T,
<em>et al.</em>
Automated Electronic Health Record to Electronic Data Capture Transfer in Clinical Studies in the German Health Care System: Feasibility Study and Gap Analysis. J Med Internet Res 2023;25:e47958. https://doi.org/10.2196/47958.
</cite> [<a href="https://doi.org/10.2196/47958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10439471/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37540555/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-43">
<span class="label">43.</span><cite>
Meystre SM, Heider PM, Cates A, Bastian G, Pittman T, Gentilin S,
<em>et al.</em>
Piloting an automated clinical trial eligibility surveillance and provider alert system based on artificial intelligence and standard data models. BMC Med Res Methodol 2023;23:88. https://doi.org/10.1186/s12874-023-01916-6.
</cite> [<a href="https://doi.org/10.1186/s12874-023-01916-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10088225/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37041475/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-44">
<span class="label">44.</span><cite>Lin J, Yu G, Gamalo M. Matching within a hybrid RCT/RWD: framework on associated causal estimands. J Biopharm Stat 2023;33:439–51. https://doi.org/10.1080/10543406.2022.2105346.</cite> [<a href="https://doi.org/10.1080/10543406.2022.2105346" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35929973/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-45">
<span class="label">45.</span><cite>
Lombardo G, Couvert C, Kose M, Begum A, Spiertz C, Worrell C,
<em>et al.</em>
Electronic health records (EHRs) in clinical research and platform trials: Application of the innovative EHR-based methods developed by EU-PEARL. J Biomed Inform 2023;148:104553. https://doi.org/10.1016/j.jbi.2023.104553.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2023.104553" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38000766/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-46">
<span class="label">46.</span><cite>Incerti D, Bretscher MT, Lin R, Harbron C. A meta-analytic framework to adjust for bias in external control studies. Pharm Stat 2023;22:162–80. https://doi.org/10.1002/pst.2266.</cite> [<a href="https://doi.org/10.1002/pst.2266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36193866/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-47">
<span class="label">47.</span><cite>
Carrigan G, Whipple S, Capra WB, Taylor MD, Brown JS, Lu M,
<em>et al.</em>
Using Electronic Health Records to Derive Control Arms for Early Phase Single-Arm Lung Cancer Trials: Proof-of-Concept in Randomized Controlled Trials. Clin Pharmacol Ther 2020;107:369–77. https://doi.org/10.1002/cpt.1586.
</cite> [<a href="https://doi.org/10.1002/cpt.1586" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7006884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31350853/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-48">
<span class="label">48.</span><cite>Pfiffner PB, Pinyol I, Natter MD, Mandl KD. C3-PRO: Connecting ResearchKit to the Health System Using i2b2 and FHIR. PloS One 2016;11:e0152722. https://doi.org/10.1371/journal.pone.0152722.</cite> [<a href="https://doi.org/10.1371/journal.pone.0152722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4816293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27031856/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-49">
<span class="label">49.</span><cite>
Sittig DF, Hazlehurst BL, Brown J, Murphy S, Rosenman M, Tarczy-Hornoch P,
<em>et al.</em>
A survey of informatics platforms that enable distributed comparative effectiveness research using multi-institutional heterogenous clinical data. Med Care 2012;50 Suppl:S49-59. https://doi.org/10.1097/MLR.0b013e318259c02b.
</cite> [<a href="https://doi.org/10.1097/MLR.0b013e318259c02b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3415281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22692259/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-50">
<span class="label">50.</span><cite>Bledsoe MJ, Grizzle WE, Clark BJ, Zeps N. Practical implementation issues and challenges for biobanks in the return of individual research results. Genet Med Off J Am Coll Med Genet 2012;14:478–83. https://doi.org/10.1038/gim.2011.67.</cite> [<a href="https://doi.org/10.1038/gim.2011.67" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3388905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22323073/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-51">
<span class="label">51.</span><cite>
O'Meara H, Carr DF, Evely J, Hobbs M, McCann G, van Staa T,
<em>et al.</em>
Electronic health records for biological sample collection: feasibility study of statin-induced myopathy using the Clinical Practice Research Datalink. Br J Clin Pharmacol 2014;77:831–8. https://doi.org/10.1111/bcp.12269.
</cite> [<a href="https://doi.org/10.1111/bcp.12269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4004403/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24308359/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-52">
<span class="label">52.</span><cite>
Forrest CB, Margolis PA, Bailey LC, Marsolo K, Del Beccaro MA, Finkelstein JA,
<em>et al.</em>
PEDSnet: a National Pediatric Learning Health System. J Am Med Inform Assoc JAMIA 2014;21:602–6. https://doi.org/10.1136/amiajnl-2014-002743.
</cite> [<a href="https://doi.org/10.1136/amiajnl-2014-002743" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4078288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24821737/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-53">
<span class="label">53.</span><cite>Fleurence RL, Curtis LH, Califf RM, Platt R, Selby JV, Brown JS. Launching PCORnet, a national patient-centered clinical research network. J Am Med Inform AssocJAMIA 2014;21:578–82. https://doi.org/10.1136/amiajnl-2014-002747.</cite> [<a href="https://doi.org/10.1136/amiajnl-2014-002747" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4078292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24821743/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-54">
<span class="label">54.</span><cite>van Ommen G-JB, Törnwall O, Bréchot C, Dagher G, Galli J, Hveem K, et al. BBMRI-ERIC as a resource for pharmaceutical and life science industries: the development of biobank-based Expert Centres. Eur J Hum Genet EJHG 2015;23:893–900. https://doi.org/10.1038/ejhg.2014.235.</cite> [<a href="https://doi.org/10.1038/ejhg.2014.235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4463510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25407005/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-55">
<span class="label">55.</span><cite>
Kass-Hout TA, Xu Z, Mohebbi M, Nelsen H, Baker A, Levine J,
<em>et al.</em>
OpenFDA: an innovative platform providing access to a wealth of FDA's publicly available data. J Am Med Inform Assoc JAMIA 2016;23:596–600. https://doi.org/10.1093/jamia/ocv153.
</cite> [<a href="https://doi.org/10.1093/jamia/ocv153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4901374/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26644398/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-56">
<span class="label">56.</span><cite>Johnson AE, Stone DJ, Celi LA, Pollard TJ. The MIMIC Code Repository: enabling reproducibility in critical care research. J Am Med Inform Assoc JAMIA 2018;25:32–9. https://doi.org/10.1093/jamia/ocx084.</cite> [<a href="https://doi.org/10.1093/jamia/ocx084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6381763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29036464/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-57">
<span class="label">57.</span><cite>
Denaxas S, Gonzalez-Izquierdo A, Direk K, Fitzpatrick NK, Fatemifar G, Banerjee A,
<em>et al.</em>
UK phenomics platform for developing and validating electronic health record phenotypes: CALIBER. J Am Med Inform Assoc JAMIA 2019;26:1545–59. https://doi.org/10.1093/jamia/ocz105.
</cite> [<a href="https://doi.org/10.1093/jamia/ocz105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6857510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31329239/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-58">
<span class="label">58.</span><cite>
Forrest CB, McTigue KM, Hernandez AF, Cohen LW, Cruz H, Haynes K,
<em>et al.</em>
PCORnet® 2020: current state, accomplishments, and future directions. J Clin Epidemiol 2021;129:60–7. https://doi.org/10.1016/j.jclinepi.2020.09.036.
</cite> [<a href="https://doi.org/10.1016/j.jclinepi.2020.09.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7521354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33002635/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-59">
<span class="label">59.</span><cite>
Raisaro JL, Marino F, Troncoso-Pastoriza J, Beau-Lejdstrom R, Bellazzi R, Murphy R,
<em>et al.</em>
SCOR: A secure international informatics infrastructure to investigate COVID-19. J Am Med Inform Assoc JAMIA 2020;27:1721–6. https://doi.org/10.1093/jamia/ocaa172.
</cite> [<a href="https://doi.org/10.1093/jamia/ocaa172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7454652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32918447/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-60">
<span class="label">60.</span><cite>
Haendel MA, Chute CG, Bennett TD, Eichmann DA, Guinney J, Kibbe WA,
<em>et al.</em>
The National COVID Cohort Collaborative (N3C): Rationale, design, infrastructure, and deployment. J Am Med Inform Assoc JAMIA 2021;28:427–43. https://doi.org/10.1093/jamia/ocaa196.
</cite> [<a href="https://doi.org/10.1093/jamia/ocaa196" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7454687/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32805036/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-61">
<span class="label">61.</span><cite>Kohane IS, Churchill SE, Murphy SN. A translational engine at the national scale: informatics for integrating biology and the bedside. J Am Med Inform Assoc JAMIA 2012;19:181–5. https://doi.org/10.1136/amiajnl-2011-000492.</cite> [<a href="https://doi.org/10.1136/amiajnl-2011-000492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3277623/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22081225/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-62">
<span class="label">62.</span><cite>MacKenzie SL, Wyatt MC, Schuff R, Tenenbaum JD, Anderson N. Practices and perspectives on building integrated data repositories: results from a 2010 CTSA survey. J Am Med Inform Assoc JAMIA 2012;19:e119-124. https://doi.org/10.1136/amiajnl-2011-000508.</cite> [<a href="https://doi.org/10.1136/amiajnl-2011-000508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3392848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22437072/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-63">
<span class="label">63.</span><cite>Jensen PB, Jensen LJ, Brunak S. Mining electronic health records: towards better research applications and clinical care. Nat Rev Genet 2012;13:395–405. https://doi.org/10.1038/nrg3208.</cite> [<a href="https://doi.org/10.1038/nrg3208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22549152/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-64">
<span class="label">64.</span><cite>
Hersh WR, Weiner MG, Embi PJ, Logan JR, Payne PRO, Bernstam EV,
<em>et al.</em>
Caveats for the use of operational electronic health record data in comparative effectiveness research. Med Care 2013;51:S30-37. https://doi.org/10.1097/MLR.0b013e31829b1dbd.
</cite> [<a href="https://doi.org/10.1097/MLR.0b013e31829b1dbd" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3748381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23774517/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-65">
<span class="label">65.</span><cite>Martin Sanchez F, Gray K, Bellazzi R, Lopez-Campos G. Exposome informatics: considerations for the design of future biomedical research information systems. J Am Med Inform Assoc JAMIA 2014;21:386–90. https://doi.org/10.1136/amiajnl-2013-001772.</cite> [<a href="https://doi.org/10.1136/amiajnl-2013-001772" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3994854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24186958/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-66">
<span class="label">66.</span><cite>Johnson EK, Broder-Fingert S, Tanpowpong P, Bickel J, Lightdale JR, Nelson CP. Use of the i2b2 research query tool to conduct a matched case-control clinical research study: advantages, disadvantages and methodological considerations. BMC Med Res Methodol 2014;14:16. https://doi.org/10.1186/1471-2288-14-16.</cite> [<a href="https://doi.org/10.1186/1471-2288-14-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3909388/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24479726/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-67">
<span class="label">67.</span><cite>
Danciu I, Cowan JD, Basford M, Wang X, Saip A, Osgood S,
<em>et al.</em>
Secondary use of clinical data: the Vanderbilt approach. J Biomed Inform 2014;52:28–35. https://doi.org/10.1016/j.jbi.2014.02.003.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2014.02.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4133331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24534443/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-68">
<span class="label">68.</span><cite>Wilhelm EE, Oster E, Shoulson I. Approaches and costs for sharing clinical research data. JAMA 2014;311:1201–2. https://doi.org/10.1001/jama.2014.850.</cite> [<a href="https://doi.org/10.1001/jama.2014.850" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24556937/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-69">
<span class="label">69.</span><cite>
Murphy SN, Herrick C, Wang Y, Wang TD, Sack D, Andriole KP,
<em>et al.</em>
High throughput tools to access images from clinical archives for research. J Digit Imaging 2015;28:194–204. https://doi.org/10.1007/s10278-014-9733-9.
</cite> [<a href="https://doi.org/10.1007/s10278-014-9733-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4359193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25316195/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-70">
<span class="label">70.</span><cite>Klann JG, Abend A, Raghavan VA, Mandl KD, Murphy SN. Data interchange using i2b2. J Am Med Inform Assoc JAMIA 2016;23:909–15. https://doi.org/10.1093/jamia/ocv188.</cite> [<a href="https://doi.org/10.1093/jamia/ocv188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4997035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26911824/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-71">
<span class="label">71.</span><cite>Jannot A-S, Zapletal E, Avillach P, Mamzer M-F, Burgun A, Degoulet P. The Georges Pompidou University Hospital Clinical Data Warehouse: A 8-years follow-up experience. Int J Med Inf 2017;102:21–8. https://doi.org/10.1016/j.ijmedinf.2017.02.006.</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2017.02.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28495345/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-72">
<span class="label">72.</span><cite>Cuggia M, Combes S. The French Health Data Hub and the German Medical Informatics Initiatives: Two National Projects to Promote Data Sharing in Healthcare. Yearb Med Inform 2019;28:195–202. https://doi.rg/10.1055/s-0039-1677917.</cite> [<a href="https://doi.org/10.1055/s-0039-1677917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6697511/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31419832/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-73">
<span class="label">73.</span><cite>
Kotecha D, Asstvelbergs FW, Achenbach S, Anker SD, Atar D, Baigent C,
<em>et al.</em>
CODE-EHR best-practice framework for the use of structured electronic health-care records in clinical research. Lancet Digit Health 2022;4:e757–64. https://doi.org/10.1016/S2589-7500(22)00151-0.
</cite> [<a href="https://doi.org/10.1016/S2589-7500(22)00151-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36050271/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-74">
<span class="label">74.</span><cite>
Ostropolets A, Albogami Y, Conover M, Banda JM, Baumgartner WA, Blacketer C,
<em>et al.</em>
Reproducible variability: assessing investigator discordance across 9 research teams attempting to reproduce the same observational study. J Am Med Inform Assoc JAMIA 2023;30:859–68. https://doi.org/10.1093/jamia/ocad009.
</cite> [<a href="https://doi.org/10.1093/jamia/ocad009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10114120/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36826399/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-75">
<span class="label">75.</span><cite>Knosp BM, Dorr DA, Campion TR. Maturity in enterprise data warehouses for research operations: Analysis of a pilot study. J Clin Transl Sci 2023;7:e70. https://doi.org/10.1017/cts.2023.23.</cite> [<a href="https://doi.org/10.1017/cts.2023.23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10052397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37008621/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-76">
<span class="label">76.</span><cite>
Geissbuhler A, Safran C, Buchan I, Bellazzi R, Labkoff S, Eilenberg K,
<em>et al.</em>
Trustworthy reuse of health data: a transnational perspective. Int J Med Inf 2013;82:1–9. https://doi.org/10.1016/j.ijmedinf.2012.11.003.
</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2012.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23182430/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-77">
<span class="label">77.</span><cite>Semler SC, Wissing F, Heyder R. German Medical Informatics Initiative. Methods Inf Med 2018;57:e50–6. https://doi.org/10.3414/ME18-03-0003.</cite> [<a href="https://doi.org/10.3414/ME18-03-0003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6178199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30016818/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-78">
<span class="label">78.</span><cite>
Sahoo SS, Lhatoo SD, Gupta DK, Cui L, Zhao M, Jayapandian C,
<em>et al.</em>
Epilepsy and seizure ontology: towards an epilepsy informatics infrastructure for clinical research and patient care. J Am Med Inform Assoc JAMIA 2014;21:82–9. https://doi.org/10.1136/amiajnl-2013-001696.
</cite> [<a href="https://doi.org/10.1136/amiajnl-2013-001696" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3912711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23686934/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-79">
<span class="label">79.</span><cite>
McMurry AJ, Murphy SN, MacFadden D, Weber G, Simons WW, Orechia J,
<em>et al.</em>
SHRINE: enabling nationally scalable multi-site disease studies. PloS One 2013;8:e55811. https://doi.org/10.1371/journal.pone.0055811.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0055811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3591385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23533569/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-80">
<span class="label">80.</span><cite>Sinaci AA, Laleci Erturkmen GB. A federated semantic metadata registry framework for enabling interoperability across clinical research and care domains. J Biomed Inform 2013;46:784–94. https://doi.org/10.1016/j.jbi.2013.05.009.</cite> [<a href="https://doi.org/10.1016/j.jbi.2013.05.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23751263/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-81">
<span class="label">81.</span><cite>
Pathak J, Bailey KR, Beebe CE, Bethard S, Carrell DC, Chen PJ,
<em>et al.</em>
Normalization and standardization of electronic health records for high-throughput phenotyping: the SHARPn consortium. J Am Med Inform Assoc JAMIA 2013;20:e341-348. https://doi.org/10.1136/amiajnl-2013-001939.
</cite> [<a href="https://doi.org/10.1136/amiajnl-2013-001939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3861933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24190931/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-82">
<span class="label">82.</span><cite>
Liang SF, Taweel A, Miles S, Kovalchuk Y, Spiridou A, Barratt B,
<em>et al.</em>
Semi automated transformation to OWL formatted files as an approach to data integration. A feasibility study using environmental, disease register and primary care clinical data. Methods Inf Med 2015;54:32–40. https://doi.org/10.3414/ME13-02-0029.
</cite> [<a href="https://doi.org/10.3414/ME13-02-0029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24903775/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-83">
<span class="label">83.</span><cite>Choquet R, Maaroufi M, de Carrara A, Messiaen C, Luigi E, Landais P. A methodology for a minimum data set for rare diseases to support national centers of excellence for healthcare and research. J Am Med Inform Assoc JAMIA 2015;22:76–85. https://doi.org/10.1136/amiajnl-2014-002794.</cite> [<a href="https://doi.org/10.1136/amiajnl-2014-002794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4433369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25038198/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-84">
<span class="label">84.</span><cite>Pang C, Hendriksen D, Dijkstra M, van der Velde KJ, Kuiper J, Hillege HL, et al. BiobankConnect: software to rapidly connect data elements for pooled analysis across biobanks using ontological and lexical indexing. J Am Med Inform Assoc JAMIA 2015;22:65–75. https://doi.org/10.1136/amiajnl-2013-002577.</cite> [<a href="https://doi.org/10.1136/amiajnl-2013-002577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4433361/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25361575/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-85">
<span class="label">85.</span><cite>Jiang G, Evans J, Oniki TA, Coyle JF, Bain L, Huff SM, et al. Harmonization of detailed clinical models with clinical study data standards. Methods Inf Med 2015;54:65–74. https://doi.org/10.3414/ME13-02-0019.</cite> [<a href="https://doi.org/10.3414/ME13-02-0019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25426730/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-86">
<span class="label">86.</span><cite>Huser V, Sastry C, Breymaier M, Idriss A, Cimino JJ. Standardizing data exchange for clinical research protocols and case report forms: An assessment of the suitability of the Clinical Data Interchange Standards Consortium (CDISC) Operational Data Model (ODM). J Biomed Inform 2015;57:88–99. https://doi.org/10.1016/j.jbi.2015.06.023.</cite> [<a href="https://doi.org/10.1016/j.jbi.2015.06.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4714951/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26188274/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-87">
<span class="label">87.</span><cite>Wilkinson MD, Dumontier M, Aalbersberg IJJ, Appleton G, Axton M, Baak A, et al. The FAIR Guiding Principles for scientific data management and stewardship. Sci Data 2016;3:160018. https://doi.org/10.1038/sdata.2016.18.</cite> [<a href="https://doi.org/10.1038/sdata.2016.18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4792175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26978244/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-88">
<span class="label">88.</span><cite>Garza M, Del Fiol G, Tenenbaum J, Walden A, Zozus MN. Evaluating common data models for use with a longitudinal community registry. J Biomed Inform 2016;64:333–41. https://doi.org/10.1016/j.jbi.2016.10.016.</cite> [<a href="https://doi.org/10.1016/j.jbi.2016.10.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6810649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27989817/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-89">
<span class="label">89.</span><cite>Hripcsak G, Ryan PB, Duke JD, Shah NH, Park RW, Huser V, et al. Characterizing treatment pathways at scale using the OHDSI network. Proc Natl Acad Sci U S A 2016;113:7329–36. https://doi.org/10.1073/pnas.1510502113.</cite> [<a href="https://doi.org/10.1073/pnas.1510502113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4941483/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27274072/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-90">
<span class="label">90.</span><cite>Alonso-Calvo R, Paraiso-Medina S, Perez-Rey D, Alonso-Oset E, van Stiphout R, Yu S, et al. A semantic interoperability approach to support integration of gene expression and clinical data in breast cancer. Comput Biol Med 2017;87:179–86. https://doi.org/10.1016/j.compbiomed.2017.06.005.</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2017.06.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28601027/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-91">
<span class="label">91.</span><cite>Yamamoto K, Ota K, Akiya I, Shintani A. A pragmatic method for transforming clinical research data from the research electronic data capture “REDCap” to Clinical Data Interchange Standards Consortium (CDISC) Study Data Tabulation Model (SDTM): Development and evaluation of REDCap2SDTM. J Biomed Inform 2017;70:65–76. https://doi.org/10.1016/j.jbi.2017.05.003.</cite> [<a href="https://doi.org/10.1016/j.jbi.2017.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28487263/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-92">
<span class="label">92.</span><cite>Paris N, Mendis M, Daniel C, Murphy S, Tannier X, Zweigenbaum P. i2b2 implemented over SMART-on-FHIR. AMIA Jt Summits Transl Sci Proc AMIA Jt Summits Transl Sci 2018;2017:369–78.</cite> [<a href="/articles/PMC5961782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29888095/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-93">
<span class="label">93.</span><cite>
Pedrera-Jiménez M, García-Barrio N, Cruz-Rojo J, Terriza-Torres AI, López-Jiménez EA, Calvo-Boyero F,
<em>et al.</em>
Obtaining EHR-derived datasets for COVID-19 research within a short time: a flexible methodology based on Detailed Clinical Models. J Biomed Inform 2021;115:103697. https://doi.org/10.1016/j.jbi.2021.103697.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2021.103697" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7857038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33548541/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-94">
<span class="label">94.</span><cite>
Phuong J, Zampino E, Dobbins N, Espinoza J, Meeker D, Spratt H,
<em>et al.</em>
Extracting Patient-level Social Determinants of Health into the OMOP Common Data Model. AMIA Annu Symp Proc AMIA Symp 2021;2021:989–98.
</cite> [<a href="/articles/PMC8861735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35308947/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-95">
<span class="label">95.</span><cite>
Gruendner J, Deppenwiese N, Folz M, Köhler T, Kroll B, Prokosch H-U,
<em>et al.</em>
The Architecture of a Feasibility Query Portal for Distributed COVID-19 Fast Healthcare Interoperability Resources (FHIR) Patient Data Repositories: Design and Implementation Study. JMIR Med Inform 2022;10:e36709. https://doi.org/10.2196/36709.
</cite> [<a href="https://doi.org/10.2196/36709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9135115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35486893/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-96">
<span class="label">96.</span><cite>
Callahan TJ, Stefanski AL, Wyrwa JM, Zeng C, Ostropolets A, Banda JM,
<em>et al.</em>
Ontologizing health systems data at scale: making translational discovery a reality. NPJ Digit Med 2023;6:89. https://doi.org/10.1038/s41746-023-00830-x.
</cite> [<a href="https://doi.org/10.1038/s41746-023-00830-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10196319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37208468/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-97">
<span class="label">97.</span><cite>Rusanov A, Weiskopf NG, Wang S, Weng C. Hidden in plain sight: bias towards sick patients when sampling patients with sufficient electronic health record data for research. BMC Med Inform Decis Mak 2014;14:51. https://doi.org/10.1186/1472-6947-14-51.</cite> [<a href="https://doi.org/10.1186/1472-6947-14-51" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4062889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24916006/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-98">
<span class="label">98.</span><cite>Johnson SG, Speedie S, Simon G, Kumar V, Westra BL. Application of An Ontology for Characterizing Data Quality For a Secondary Use of EHR Data. Appl Clin Inform 2016;7:69–88. https://doi.org/10.4338/ACI-2015-08-RA-0107.</cite> [<a href="https://doi.org/10.4338/ACI-2015-08-RA-0107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4817336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27081408/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-99">
<span class="label">99.</span><cite>
Kahn MG, Callahan TJ, Barnard J, Bauck AE, Brown J, Davidson BN,
<em>et al.</em>
A Harmonized Data Quality Assessment Terminology and Framework for the Secondary Use of Electronic Health Record Data. EGEMS Wash DC 2016;4:1244. https://doi.org/10.13063/2327-9214.1244.
</cite> [<a href="https://doi.org/10.13063/2327-9214.1244" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5051581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27713905/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-100">
<span class="label">100.</span><cite>Estiri H, Stephens KA, Klann JG, Murphy SN. Exploring completeness in clinical data research networks with DQe-c. J Am Med Inform Assoc JAMIA 2018;25:17–24. https://doi.org/10.1093/jamia/ocx109.</cite> [<a href="https://doi.org/10.1093/jamia/ocx109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6481389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29069394/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-101">
<span class="label">101.</span><cite>Qualls LG, Phillips TA, Hammill BG, Topping J, Louzao DM, Brown JS, et al. Evaluating Foundational Data Quality in the National Patient-Centered Clinical Research Network (PCORnet®). EGEMS Wash DC 2018;6:3. https://doi.org/10.5334/egems.199.</cite> [<a href="https://doi.org/10.5334/egems.199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5983028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29881761/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-102">
<span class="label">102.</span><cite>Daniel C, Serre P, Orlova N, Bréant S, Paris N, Griffon N. Initializing a hospital-wide data quality program. The AP-HP experience. Comput Methods Programs Biomed 2019;181:104804. https://doi.org/10.1016/j.cmpb.2018.10.016.</cite> [<a href="https://doi.org/10.1016/j.cmpb.2018.10.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30497872/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-103">
<span class="label">103.</span><cite>
Bell SK, Delbanco T, Elmore JG, Fitzgerald PS, Fossa A, Harcourt K,
<em>et al.</em>
Frequency and Types of Patient-Reported Errors in Electronic Health Record Ambulatory Care Notes. JAMA Netw Open 2020;3:e205867. https://doi.org/10.1001/amanetworkopen.2020.5867.
</cite> [<a href="https://doi.org/10.1001/jamanetworkopen.2020.5867" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7284300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32515797/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-104">
<span class="label">104.</span><cite>
Bian J, Lyu T, Loiacono A, Viramontes TM, Lipori G, Guo Y,
<em>et al.</em>
Assessing the practice of data quality evaluation in a national clinical data research network through a systematic scoping review in the era of real-world data. J Am Med Inform Assoc JAMIA 2020;27:1999–2010. https://doi.org/10.1093/jamia/ocaa245.
</cite> [<a href="https://doi.org/10.1093/jamia/ocaa245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7727392/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33166397/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-105">
<span class="label">105.</span><cite>Mathes T, Rombey T, Kuss O, Pieper D. No inexplicable disagreements between real-world data-based nonrandomized controlled studies and randomized controlled trials were found. J Clin Epidemiol 2021;133:1–13. https://doi.org/10.1016/j.jclinepi.2020.12.019.</cite> [<a href="https://doi.org/10.1016/j.jclinepi.2020.12.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33359322/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-106">
<span class="label">106.</span><cite>Blacketer C, Defalco FJ, Ryan PB, Rijnbeek PR. Increasing trust in real-world evidence through evaluation of observational data quality. J Am Med Inform Assoc JAMIA 2021;28:2251–7. https://doi.org/10.1093/jamia/ocab132.</cite> [<a href="https://doi.org/10.1093/jamia/ocab132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8449628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34313749/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-107">
<span class="label">107.</span><cite>Tahar K, Martin T, Mou Y, Verbuecheln R, Graessner H, Krefting D. Rare Diseases in Hospital Information Systems-An Interoperable Methodology for Distributed Data Quality Assessments. Methods Inf Med 2023;62:71–89. https://doi.org/10.1055/a-2006-1018.</cite> [<a href="https://doi.org/10.1055/a-2006-1018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10462432/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36596461/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-108">
<span class="label">108.</span><cite>
Syed R, Eden R, Makasi T, Chukwudi I, Mamudu A, Kamalpour M,
<em>et al.</em>
Digital Health Data Quality Issues: Systematic Review. J Med Internet Res 2023;25:e42615. https://doi.org/10.2196/42615.
</cite> [<a href="https://doi.org/10.2196/42615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10131725/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37000497/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-109">
<span class="label">109.</span><cite>Lee S, Roh G-H, Kim J-Y, Ho Lee Y, Woo H, Lee S. Effective data quality management for electronic medical record data using SMART DATA. Int J Med Inf 2023;180:105262. https://doi.org/10.1016/j.ijmedinf.2023.105262.</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2023.105262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37871445/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-110">
<span class="label">110.</span><cite>Declerck J, Kalra D, Vander Stichele R, Coorevits P. Frameworks, Dimensions, Definitions of Aspects, and Assessment Methods for the Appraisal of Quality of Health Data for Secondary Use: Comprehensive Overview of Reviews. JMIR Med Inform 2024;12:e51560. https://doi.org/10.2196/51560.</cite> [<a href="https://doi.org/10.2196/51560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10955383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38446534/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-111">
<span class="label">111.</span><cite>Eder J, Gottweis H, Zatloukal K. IT solutions for privacy protection in biobanking. Public Health Genomics 2012;15:254–62. https://doi.org/10.1159/000336663.</cite> [<a href="https://doi.org/10.1159/000336663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22722689/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-112">
<span class="label">112.</span><cite>Atreya RV, Smith JC, McCoy AB, Malin B, Miller RA. Reducing patient re-identification risk for laboratory results within research datasets. J Am Med Inform Assoc JAMIA 2013;20:95–101. https://doi.org/10.1136/amiajnl-2012-001026.</cite> [<a href="https://doi.org/10.1136/amiajnl-2012-001026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3555327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22822040/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-113">
<span class="label">113.</span><cite>
Kho AN, Cashy JP, Jackson KL, Pah AR, Goel S, Boehnke J,
<em>et al.</em>
Design and implementation of a privacy preserving electronic health record linkage tool in Chicago. J Am Med Inform Assoc JAMIA 2015;22:1072–80. https://doi.org/10.1093/jamia/ocv038.
</cite> [<a href="https://doi.org/10.1093/jamia/ocv038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5009931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26104741/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-114">
<span class="label">114.</span><cite>Harmanci A, Gerstein M. Quantification of private information leakage from phenotype-genotype data: linking attacks. Nat Methods 2016;13:251–6. https://doi.org/10.1038/nmeth.3746.</cite> [<a href="https://doi.org/10.1038/nmeth.3746" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4834871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26828419/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-115">
<span class="label">115.</span><cite>Prasser F, Spengler H, Bild R, Eicher J, Kuhn KA. Privacy-enhancing ETL-processes for biomedical data. Int J Med Inf 2019;126:72–81. https://doi.org/10.1016/j.ijmedinf.2019.03.006.</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2019.03.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31029266/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-116">
<span class="label">116.</span><cite>Zhang Z, Yan C, Mesa DA, Sun J, Malin BA. Ensuring electronic medical record simulation through better training, modeling, and evaluation. J Am Med Inform Assoc JAMIA 2020;27:99–108. https://doi.org/10.1093/jamia/ocz161.</cite> [<a href="https://doi.org/10.1093/jamia/ocz161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6913223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31592533/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-117">
<span class="label">117.</span><cite>Paddock S, Abedtash H, Zummo J, Thomas S. Proof-of-concept study: Homomorphically encrypted data can support real-time learning in personalized cancer medicine. BMC Med Inform Decis Mak 2019;19:255. https://doi.org/10.1186/s12911-019-0983-9.</cite> [<a href="https://doi.org/10.1186/s12911-019-0983-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6894133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31801535/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-118">
<span class="label">118.</span><cite>
Foraker RE, Yu SC, Gupta A, Michelson AP, Pineda Soto JA, Colvin R,
<em>et al.</em>
Spot the difference: comparing results of analyses from real patient data and synthetic derivatives. JAMIA Open 2020;3:557–66. https://doi.org/10.1093/jamiaopen/ooaa060.
</cite> [<a href="https://doi.org/10.1093/jamiaopen/ooaa060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7886551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33623891/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-119">
<span class="label">119.</span><cite>
Marsolo K, Kiernan D, Toh S, Phua J, Louzao D, Haynes K,
<em>et al.</em>
Assessing the impact of privacy-preserving record linkage on record overlap and patient demographic and clinical characteristics in PCORnet®, the National Patient-Centered Clinical Research Network. J Am Med Inform Assoc JAMIA 2023;30:447–55. https://doi.org/10.1093/jamia/ocac229.
</cite> [<a href="https://doi.org/10.1093/jamia/ocac229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9933062/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36451264/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-120">
<span class="label">120.</span><cite>Hripcsak G, Albers DJ. Next-generation phenotyping of electronic health records. J Am Med Inform Assoc JAMIA 2013;20:117–21. https://doi.org/10.1136/amiajnl-2012-001145.</cite> [<a href="https://doi.org/10.1136/amiajnl-2012-001145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3555337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22955496/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-121">
<span class="label">121.</span><cite>
Newton KM, Peissig PL, Kho AN, Bielinski SJ, Berg RL, Choudhary V,
<em>et al.</em>
Validation of electronic medical record-based phenotyping algorithms: results and lessons learned from the eMERGE network. J Am Med Inform Assoc JAMIA 2013;20:e147-154. https://doi.org/10.1136/amiajnl-2012-000896.
</cite> [<a href="https://doi.org/10.1136/amiajnl-2012-000896" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3715338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23531748/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-122">
<span class="label">122.</span><cite>
Kirby JC, Speltz P, Rasmussen LV, Basford M, Gottesman O, Peissig PL,
<em>et al.</em>
PheKB: a catalog and workflow for creating electronic phenotype algorithms for transportability. J Am Med Inform Assoc JAMIA 2016;23:1046–52. https://doi.org/10.1093/jamia/ocv202.
</cite> [<a href="https://doi.org/10.1093/jamia/ocv202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5070514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27026615/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-123">
<span class="label">123.</span><cite>Pivovarov R, Perotte AJ, Grave E, Angiolillo J, Wiggins CH, Elhadad N. Learning probabilistic phenotypes from heterogeneous EHR data. J Biomed Inform 2015;58:156–65. https://doi.org/10.1016/j.jbi.2015.10.001.</cite> [<a href="https://doi.org/10.1016/j.jbi.2015.10.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8025140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26464024/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-124">
<span class="label">124.</span><cite>Halpern Y, Horng S, Choi Y, Sontag D. Electronic medical record phenotyping using the anchor and learn framework. J Am Med Inform Assoc JAMIA 2016;23:731–40. https://doi.org/10.1093/jamia/ocw011.</cite> [<a href="https://doi.org/10.1093/jamia/ocw011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4926745/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27107443/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-125">
<span class="label">125.</span><cite>
Agarwal V, Podchiyska T, Banda JM, Goel V, Leung TI, Minty EP,
<em>et al.</em>
Learning statistical models of phenotypes using noisy labeled training data. J Am Med Inform Assoc JAMIA 2016;23:1166–73. https://doi.org/10.1093/jamia/ocw028.
</cite> [<a href="https://doi.org/10.1093/jamia/ocw028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5070523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27174893/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-126">
<span class="label">126.</span><cite>
Luo L, Li L, Hu J, Wang X, Hou B, Zhang T,
<em>et al.</em>
A hybrid solution for extracting structured medical information from unstructured data in medical records via a double-reading/entry system. BMC Med Inform Decis Mak 2016;16:114. https://doi.org/10.1186/s12911-016-0357-5.
</cite> [<a href="https://doi.org/10.1186/s12911-016-0357-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5006527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27577240/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-127">
<span class="label">127.</span><cite>Brisimi TS, Chen R, Mela T, Olshevsky A, Paschalidis IC, Shi W. Federated learning of predictive models from federated Electronic Health Records. Int J Med Inf 2018;112:59–67. https://doi.org/10.1016/j.ijmedinf.2018.01.007.</cite> [<a href="https://doi.org/10.1016/j.ijmedinf.2018.01.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5836813/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29500022/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-128">
<span class="label">128.</span><cite>Agniel D, Kohane IS, Weber GM. Biases in electronic health record data due to processes within the healthcare system: retrospective observational study. BMJ 2018;361:k1479. https://doi.org/10.1136/bmj.k1479.</cite> [<a href="https://doi.org/10.1136/bmj.k1479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5925441/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29712648/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-129">
<span class="label">129.</span><cite>
Zhang Y, Cai T, Yu S, Cho K, Hong C, Sun J,
<em>et al.</em>
High-throughput phenotyping with electronic medical record data using a common semi-supervised approach (PheCAP). Nat Protoc 2019;14:3426–44. https://doi.org/10.1038/s41596-019-0227-6.
</cite> [<a href="https://doi.org/10.1038/s41596-019-0227-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7323894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31748751/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-130">
<span class="label">130.</span><cite>
Kashyap M, Seneviratne M, Banda JM, Falconer T, Ryu B, Yoo S,
<em>et al.</em>
Development and validation of phenotype classifiers across multiple sites in the observational health data sciences and informatics network. J Am Med Inform Assoc JAMIA 2020;27:877–83. https://doi.org/10.1093/jamia/ocaa032.
</cite> [<a href="https://doi.org/10.1093/jamia/ocaa032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7309227/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32374408/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-131">
<span class="label">131.</span><cite>Eaneff S, Obermeyer Z, Butte AJ. The Case for Algorithmic Stewardship for Artificial Intelligence and Machine Learning Technologies. JAMA 2020;324:1397–8. https://doi.org/10.1001/jama.2020.9371.</cite> [<a href="https://doi.org/10.1001/jama.2020.9371" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32926087/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-132">
<span class="label">132.</span><cite>Estiri H, Strasser ZH, Murphy SN. High-throughput phenotyping with temporal sequences. J Am Med Inform Assoc JAMIA 2021;28:772–81. https://doi.org/10.1093/jamia/ocaa288.</cite> [<a href="https://doi.org/10.1093/jamia/ocaa288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7973443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33313899/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-133">
<span class="label">133.</span><cite>
Bedoya AD, Economou-Zavlanos NJ, Goldstein BA, Young A, Jelovsek JE, O'Brien C,
<em>et al.</em>
A framework for the oversight and local deployment of safe and high-quality prediction models. J Am Med Inform Assoc JAMIA 2022;29:1631–6. https://doi.org/10.1093/jamia/ocac078.
</cite> [<a href="https://doi.org/10.1093/jamia/ocac078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9382367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35641123/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-134">
<span class="label">134.</span><cite>Ahuja Y, Zou Y, Verma A, Buckeridge D, Li Y. MixEHR-Guided: A guided multi-modal topic modeling approach for large-scale automatic phenotyping using the electronic health record. J Biomed Inform 2022;134:104190. https://doi.org/10.1016/j.jbi.2022.104190.</cite> [<a href="https://doi.org/10.1016/j.jbi.2022.104190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36058522/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-135">
<span class="label">135.</span><cite>
Keloth VK, Banda JM, Gurley M, Heider PM, Kennedy G, Liu H,
<em>et al.</em>
Representing and utilizing clinical textual data for real world studies: An OHDSI approach. J Biomed Inform 2023;142:104343. https://doi.org/10.1016/j.jbi.2023.104343.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2023.104343" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10428170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36935011/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-136">
<span class="label">136.</span><cite>Hamidi B, Flume PA, Simpson KN, Alekseyenko AV. Not all phenotypes are created equal: covariates of success in e-phenotype specification. J Am Med Inform Assoc JAMIA 2023;30:213–21. https://doi.org/10.1093/jamia/ocac157.</cite> [<a href="https://doi.org/10.1093/jamia/ocac157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9846689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36069977/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-137">
<span class="label">137.</span><cite>
Brandt PS, Kho A, Luo Y, Pacheco JA, Walunas TL, Hakonarson H,
<em>et al.</em>
Characterizing variability of electronic health record-driven phenotype definitions. J Am Med Inform Assoc JAMIA 2023;30:427–37. https://doi.org/10.1093/jamia/ocac235.
</cite> [<a href="https://doi.org/10.1093/jamia/ocac235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9933077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36474423/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-138">
<span class="label">138.</span><cite>Huser V, Cimino JJ. Linking ClinicalTrials.gov and PubMed to track results of interventional human clinical trials. PloS One 2013;8:e68409. https://doi.org/10.1371/journal.pone.0068409.</cite> [<a href="https://doi.org/10.1371/journal.pone.0068409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3706420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23874614/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-139">
<span class="label">139.</span><cite>Heneberg P. From Excessive Journal Self-Cites to Citation Stacking: Analysis of Journal Self-Citation Kinetics in Search for Journals, Which Boost Their Scientometric Indicators. PloS One 2016;11:e0153730. https://doi.org/10.1371/journal.pone.0153730.</cite> [<a href="https://doi.org/10.1371/journal.pone.0153730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4835057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27088862/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-140">
<span class="label">140.</span><cite>
Kohane IS, Aronow BJ, Avillach P, Beaulieu-Jones BK, Bellazzi R, Bradford RL,
<em>et al.</em>
What Every Reader Should Know About Studies Using Electronic Health Record Data but May Be Afraid to Ask. J Med Internet Res 2021;23:e22219. https://doi.org/10.2196/22219.
</cite> [<a href="https://doi.org/10.2196/22219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7927948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33600347/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-141">
<span class="label">141.</span><cite>
McIntosh LD, Juehne A, Vitale CRH, Liu X, Alcoser R, Lukas JC,
<em>et al.</em>
Repeat: a framework to assess empirical reproducibility in biomedical research. BMC Med Res Methodol 2017;17:143. https://doi.org/10.1186/s12874-017-0377-6.
</cite> [<a href="https://doi.org/10.1186/s12874-017-0377-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5604503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28923006/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-142">
<span class="label">142.</span><cite>Slager SL, Weir CR, Kim H, Mostafa J, Del Fiol G. Physicians' perception of alternative displays of clinical research evidence for clinical decision support - A study with case vignettes. J Biomed Inform 2017;71S:S53–9. https://doi.org/10.1016/j.jbi.2017.01.007.</cite> [<a href="https://doi.org/10.1016/j.jbi.2017.01.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5509533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28089913/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-143">
<span class="label">143.</span><cite>
Rea S, Pathak J, Savova G, Oniki TA, Westberg L, Beebe CE,
<em>et al.</em>
Building a robust, scalable and standards-driven infrastructure for secondary use of EHR data: the SHARPn project. J Biomed Inform 2012;45:763–71. https://doi.org/10.1016/j.jbi.2012.01.009.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2012.01.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4905766/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22326800/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-144">
<span class="label">144.</span><cite>O'Driscoll A, Daugelaite J, Sleator RD. “Big data”, Hadoop and cloud computing in genomics. J Biomed Inform 2013;46:774–81. https://doi.org/10.1016/j.jbi.2013.07.001.</cite> [<a href="https://doi.org/10.1016/j.jbi.2013.07.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23872175/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-145">
<span class="label">145.</span><cite>
Sahoo SS, Jayapandian C, Garg G, Kaffashi F, Chung S, Bozorgi A,
<em>et al.</em>
Heart beats in the cloud: distributed analysis of electrophysiological “Big Data” using cloud computing for epilepsy clinical research. J Am Med Inform Assoc JAMIA 2014;21:263–71. https://doi.org/10.1136/amiajnl-2013-002156.
</cite> [<a href="https://doi.org/10.1136/amiajnl-2013-002156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3932471/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24326538/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-146">
<span class="label">146.</span><cite>Bellazzi R, Dagliati A, Sacchi L, Segagni D. Big Data Technologies: New Opportunities for Diabetes Management. J Diabetes Sci Technol 2015;9:1119–25. https://doi.177/1932296815583505.</cite> [<a href="https://doi.org/10.1177/1932296815583505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4667334/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25910540/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-147">
<span class="label">147.</span><cite>
Schüssler-Fiorenza Rose SM, Contrepois K, Moneghetti KJ, Zhou W, Mishra T, Mataraso S,
<em>et al.</em>
A longitudinal big data approach for precision health. Nat Med 2019;25:792–804. https://doi.org/10.1038/s41591-019-0414-6.
</cite> [<a href="https://doi.org/10.1038/s41591-019-0414-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6713274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31068711/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-148">
<span class="label">148.</span><cite>
Bahmani A, Alavi A, Buergel T, Upadhyayula S, Wang Q, Ananthakrishnan SK,
<em>et al.</em>
A scalable, secure, and interoperable platform for deep data-driven health management. Nat Commun 2021;12:5757. https://doi.org/10.1038/s41467-021-26040-1.
</cite> [<a href="https://doi.org/10.1038/s41467-021-26040-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8486823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34599181/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-149">
<span class="label">149.</span><cite>
Brat GA, Weber GM, Gehlenborg N, Avillach P, Palmer NP, Chiovato L,
<em>et al.</em>
International electronic health record-derived COVID-19 clinical course profiles: the 4CE consortium. NPJ Digit Med 2020;3:109. https://doi.org/10.1038/s41746-020-00308-0.
</cite> [<a href="https://doi.org/10.1038/s41746-020-00308-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7438496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32864472/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-150">
<span class="label">150.</span><cite>Kim SYH, Miller FG. Informed consent for pragmatic trials--the integrated consent model. N Engl J Med 2014;370:769–72. https://doi.org/10.1056/NEJMhle1312508.</cite> [<a href="https://doi.org/10.1056/NEJMhle1312508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24552326/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-151">
<span class="label">151.</span><cite>Sugarman J, Califf RM. Ethics and regulatory complexities for pragmatic clinical trials. JAMA 2014;311:2381–2. https://doi.org/10.1001/jama.2014.4164.</cite> [<a href="https://doi.org/10.1001/jama.2014.4164" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24810723/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-152">
<span class="label">152.</span><cite>Gray EA, Thorpe JH. Comparative effectiveness research and big data: balancing potential with legal and ethical considerations. J Comp Eff Res 2015;4:61–74. https://doi.org/10.2217/cer.14.51.</cite> [<a href="https://doi.org/10.2217/cer.14.51" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25565069/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-153">
<span class="label">153.</span><cite>
Kim H, Bell E, Kim J, Sitapati A, Ramsdell J, Farcas C,
<em>et al.</em>
iCONCUR: informed consent for clinical data and bio-sample use for research. J Am Med Inform Assoc JAMIA 2017;24:380–7. https://doi.org/10.1093/jamia/ocw115.
</cite> [<a href="https://doi.org/10.1093/jamia/ocw115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5391727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27589942/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-154">
<span class="label">154.</span><cite>Lea NC, Nicholls J, Fitzpatrick NK. Between Scylla and Charybdis: Charting the Wicked Problem of Reusing Health Data for Clinical Research Informatics. Yearb Med Inform 2018;27:170–6. https://doi.rg/10.1055/s-0038-1641219.</cite> [<a href="https://doi.org/10.1055/s-0038-1641219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6115213/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30157520/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-155">
<span class="label">155.</span><cite>
Chandler R, Brady KT, Jerome RN, Eder M, Rothwell E, Brownley KA,
<em>et al.</em>
Broad-scale informed consent: A survey of the CTSA landscape. J Clin Transl Sci 2019;3:253–60. https://doi.org/10.1017/cts.2019.397.
</cite> [<a href="https://doi.org/10.1017/cts.2019.397" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6813518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31660250/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-156">
<span class="label">156.</span><cite>
Ferretti A, Ienca M, Sheehan M, Blasimme A, Dove ES, Farsides B,
<em>et al.</em>
Ethics review of big data research: What should stay and what should be reformed? BMC Med Ethics 2021;22:51. https://doi.org/10.1186/s12910-021-00616-4.
</cite> [<a href="https://doi.org/10.1186/s12910-021-00616-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8085804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33931049/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-157">
<span class="label">157.</span><cite>Christofidou M, Lea N, Coorevits P. A Literature Review on the GDPR, COVID-19 and the Ethical Considerations of Data Protection During a Time of Crisis. Yearb Med Inform 2021;30:226–32. https://doi.rg/10.1055/s-0041-1726512.</cite> [<a href="https://doi.org/10.1055/s-0041-1726512" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8416196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34479394/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-158">
<span class="label">158.</span><cite>Kargl M, Plass M, Müller H. A Literature Review on Ethics for AI in Biomedical Research and Biobanking. Yearb Med Inform 2022;31:152–60. https://doi.rg/10.1055/s-0042-1742516.</cite> [<a href="https://doi.org/10.1055/s-0042-1742516" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9719772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36463873/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-159">
<span class="label">159.</span><cite>Larson DB, Harvey H, Rubin DL, Irani N, Tse JR, Langlotz CP. Regulatory Frameworks for Development and Evaluation of Artificial Intelligence-Based Diagnostic Imaging Algorithms: Summary and Recommendations. J Am Coll Radiol JACR 2021;18:413–24. https://doi.org/10.1016/j.jacr.2020.09.060.</cite> [<a href="https://doi.org/10.1016/j.jacr.2020.09.060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7574690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33096088/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-160">
<span class="label">160.</span><cite>
Solomonides AE, Koski E, Atabaki SM, Weinberg S, McGreevey JD, Kannry JL,
<em>et al.</em>
Defining AMIA's artificial intelligence principles. J Am Med Inform Assoc JAMIA 2022;29:585–91. https://doi.org/10.1093/jamia/ocac006.
</cite> [<a href="https://doi.org/10.1093/jamia/ocac006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8922174/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35190824/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-161">
<span class="label">161.</span><cite>Ohno-Machado L. NIH's Big Data to Knowledge initiative and the advancement of biomedical informatics. J Am Med Inform Assoc JAMIA 2014;21:193. https://doi.org/10.1136/amiajnl-2014-002666.</cite> [<a href="https://doi.org/10.1136/amiajnl-2014-002666" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3932475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24509598/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-162">
<span class="label">162.</span><cite>
Holmes JH, Elliott TE, Brown JS, Raebel MA, Davidson A, Nelson AF,
<em>et al.</em>
Clinical research data warehouse governance for distributed research networks in the USA: a systematic review of the literature. J Am Med Inform Assoc JAMIA 2014;21:730–6. https://doi.org/10.1136/amiajnl-2013-002370.
</cite> [<a href="https://doi.org/10.1136/amiajnl-2013-002370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4078282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24682495/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-163">
<span class="label">163.</span><cite>Wilcox A, Randhawa G, Embi P, Cao H, Kuperman GJ. Sustainability considerations for health research and analytic data infrastructures. EGEMS Wash DC 2014;2:1113. https://doi.org/10.13063/2327-9214.1113.</cite> [<a href="https://doi.org/10.13063/2327-9214.1113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4371522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25848610/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-164">
<span class="label">164.</span><cite>
Unertl KM, Schaefbauer CL, Campbell TR, Senteio C, Siek KA, Bakken S,
<em>et al.</em>
Integrating community-based participatory research and informatics approaches to improve the engagement and health of underserved populations. J Am Med Inform Assoc JAMIA 2016;23:60–73. https://doi.org/10.1093/jamia/ocv094.
</cite> [<a href="https://doi.org/10.1093/jamia/ocv094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4713901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26228766/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-165">
<span class="label">165.</span><cite>Longo DL, Drazen JM. Data Sharing. N Engl J Med 2016;374:276–7. https://doi.org/10.1056/NEJMe1516564.</cite> [<a href="https://doi.org/10.1056/NEJMe1516564" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26789876/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-166">
<span class="label">166.</span><cite>
Wiley LK, Tarczy-Hornoch P, Denny JC, Freimuth RR, Overby CL, Shah N,
<em>et al.</em>
Harnessing next-generation informatics for personalizing medicine: a report from AMIA's 2014 Health Policy Invitational Meeting. J Am Med Inform Assoc JAMIA 2016;23:413–9. https://doi.org/10.1093/jamia/ocv111.
</cite> [<a href="https://doi.org/10.1093/jamia/ocv111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6457095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26911808/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-167">
<span class="label">167.</span><cite>McCray AT, Glaser J, Koppel R, Langlotz CP, Silverstein J. Health IT vendors and the academic community: The 2014 ACMI debate. J Biomed Inform 2016;60:365–75. https://doi.org/10.1016/j.jbi.2016.03.003.</cite> [<a href="https://doi.org/10.1016/j.jbi.2016.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26968349/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-168">
<span class="label">168.</span><cite>Spencer K, Sanders C, Whitley EA, Lund D, Kaye J, Dixon WG. Patient Perspectives on Sharing Anonymized Personal Health Data Using a Digital System for Dynamic Consent and Research Feedback: A Qualitative Study. J Med Internet Res 2016;18:e66. https://doi.org/10.2196/jmir.5011.</cite> [<a href="https://doi.org/10.2196/jmir.5011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4851723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27083521/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-169">
<span class="label">169.</span><cite>Valenta AL, Meagher EA, Tachinardi U, Starren J. Core informatics competencies for clinical and translational scientists: what do our customers and collaborators need to know? J Am Med Inform Assoc JAMIA 2016;23:835–9. https://doi.org/10.1093/jamia/ocw047.</cite> [<a href="https://doi.org/10.1093/jamia/ocw047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4926750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27121608/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-170">
<span class="label">170.</span><cite>
Snyder CF, Smith KC, Bantug ET, Tolbert EE, Blackford AL, Brundage MD,
<em>et al.</em>
What do these scores mean? Presenting patient-reported outcomes data to patients and clinicians to improve interpretability. Cancer 2017;123:1848–59. https://doi.org/10.1002/cncr.30530.
</cite> [<a href="https://doi.org/10.1002/cncr.30530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5419857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28085201/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-171">
<span class="label">171.</span><cite>Adler-Milstein J, Embi PJ, Middleton B, Sarkar IN, Smith J. Crossing the health IT chasm: considerations and policy recommendations to overcome current challenges and enable value-based care. J Am Med Inform Assoc JAMIA 2017;24:1036–43. https://doi.org/10.1093/jamia/ocx017.</cite> [<a href="https://doi.org/10.1093/jamia/ocx017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7651968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28340128/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-172">
<span class="label">172.</span><cite>
Richesson RL, Green BB, Laws R, Puro J, Kahn MG, Bauck A,
<em>et al.</em>
Pragmatic (trial) informatics: a perspective from the NIH Health Care Systems Research Collaboratory. J Am Med Inform Assoc JAMIA 2017;24:996–1001. https://doi.org/10.1093/jamia/ocx016.
</cite> [<a href="https://doi.org/10.1093/jamia/ocx016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6916747/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28340241/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-173">
<span class="label">173.</span><cite>Sydes MR, Ashby D. Data Authorship as an Incentive to Data Sharing. N Engl J Med 2017;377:402. https://doi.org/10.1056/NEJMc1707245.</cite> [<a href="https://doi.org/10.1056/NEJMc1707245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28748690/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-174">
<span class="label">174.</span><cite>Sanchez-Pinto LN, Mosa ASM, Fultz-Hollis K, Tachinardi U, Barnett WK, Embi PJ. The Emerging Role of the Chief Research Informatics Officer in Academic Health Centers. Appl Clin Inform 2017;8:845–53. https://doi.org/10.4338/ACI-2017-04-RA-0062.</cite> [<a href="https://doi.org/10.4338/ACI-2017-04-RA-0062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6220697/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28832068/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-175">
<span class="label">175.</span><cite>
Franklin P, Chenok K, Lavalee D, Love R, Paxton L, Segal C,
<em>et al.</em>
Framework To Guide The Collection And Use Of Patient-Reported Outcome Measures In The Learning Healthcare System. EGEMS Wash DC 2017;5:17. https://doi.org/10.5334/egems.227.
</cite> [<a href="https://doi.org/10.5334/egems.227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5983040/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29881737/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-176">
<span class="label">176.</span><cite>Beier K, Schweda M, Schicktanz S. Taking patient involvement seriously: a critical ethical analysis of participatory approaches in data-intensive medical research. BMC Med Inform Decis Mak 2019;19:90. https://doi.org/10.1186/s12911-019-0799-7.</cite> [<a href="https://doi.org/10.1186/s12911-019-0799-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6482526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31023321/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-177">
<span class="label">177.</span><cite>
Kalra D, Stroetmann V, Sundgren M, Dupont D, Schlünder I, Thienpont G,
<em>et al.</em>
The European Institute for Innovation through Health Data. Learn Health Syst 2017;1:e10008. https://doi.org/10.1002/lrh2.10008.
</cite> [<a href="https://doi.org/10.1002/lrh2.10008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6516723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31245550/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-178">
<span class="label">178.</span><cite>
Embi PJ, Richesson R, Tenenbaum J, Kannry J, Friedman C, Sarkar IN,
<em>et al.</em>
Reimagining the research-practice relationship: policy recommendations for informatics-enabled evidence-generation across the US health system. JAMIA Open 2019;2:2–9. https://doi.org/10.1093/jamiaopen/ooy056.
</cite> [<a href="https://doi.org/10.1093/jamiaopen/ooy056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6951885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31984339/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-179">
<span class="label">179.</span><cite>Zayas-Cabán T, Chaney KJ, Rucker DW. National health information technology priorities for research: A policy and development agenda. J Am Med Inform Assoc JAMIA 2020;27:652–7. https://doi.org/10.1093/jamia/ocaa008.</cite> [<a href="https://doi.org/10.1093/jamia/ocaa008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7647301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32090265/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-180">
<span class="label">180.</span><cite>
Byrne LM, Cook SK, Kennedy N, Russell M, Jerome RN, Tan J,
<em>et al.</em>
Opening doors to clinical trial participation among Hispanics: Lessons learned from the Spanish translation of ResearchMatch. J Clin Transl Sci 2020;5:e46. https://doi.org/10.1017/cts.2020.539.
</cite> [<a href="https://doi.org/10.1017/cts.2020.539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8057389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33948268/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-181">
<span class="label">181.</span><cite>
Zayas-Cabán T, Abernethy AP, Brennan PF, Devaney S, Kerlavage AR, Ramoni R,
<em>et al.</em>
Leveraging the health information technology infrastructure to advance federal research priorities. J Am Med Inform Assoc JAMIA 2020;27:647–51. https://doi.org/10.1093/jamia/ocaa011.
</cite> [<a href="https://doi.org/10.1093/jamia/ocaa011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7647255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32090259/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-182">
<span class="label">182.</span><cite>
Zenker S, Strech D, Ihrig K, Jahns R, Müller G, Schickhardt C,
<em>et al.</em>
Data protection-compliant broad consent for secondary use of health care data and human biosamples for (bio)medical research: Towards a new German national standard. J Biomed Inform 2022;131:104096. https://doi.org/10.1016/j.jbi.2022.104096.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2022.104096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35643273/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-183">
<span class="label">183.</span><cite>
Peters AE, Ogunniyi MO, Hegde SM, Bianco C, Ghafghazi S, Hernandez AF,
<em>et al.</em>
A multicenter program for electronic health record screening for patients with heart failure with preserved ejection fraction: Lessons from the DELIVER-EHR initiative. Contemp Clin Trials 2022;121:106924. https://doi.org/10.1016/j.cct.2022.106924.
</cite> [<a href="https://doi.org/10.1016/j.cct.2022.106924" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36100197/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-184">
<span class="label">184.</span><cite>Maurud S, Henni SH, Moen A. Health Equity in Clinical Research Informatics. Yearb Med Inform 2023;32:138–45. https://doi.rg/10.1055/s-0043-1768720.</cite> [<a href="https://doi.org/10.1055/s-0043-1768720" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10751137/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37414033/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-185">
<span class="label">185.</span><cite>
Alberto IRI, Alberto NRI, Ghosh AK, Jain B, Jayakumar S, Martinez-Martin N,
<em>et al.</em>
The impact of commercial health datasets on medical research and health-care algorithms. Lancet Digit Health 2023;5:e288–94. https://doi.org/10.1016/S2589-7500(23)00025-0.
</cite> [<a href="https://doi.org/10.1016/S2589-7500(23)00025-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10155113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37100543/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-186">
<span class="label">186.</span><cite>
Russell ES, Aubrun E, Moga DC, Guedes S, Camelo Castillo W, Hardy JR,
<em>et al.</em>
FDA draft guidance to improve clinical trial diversity: Opportunities for pharmacoepidemiology. J Clin Transl Sci 2023;7:e101. https://doi.org/10.1017/cts.2023.515.
</cite> [<a href="https://doi.org/10.1017/cts.2023.515" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10225256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37251002/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-187">
<span class="label">187.</span><cite>Royce TJ, Zhao Y, Ryals CA. Improving Diversity in Clinical Trials by Using Real-world Data to Define Eligibility Criteria. JAMA Oncol 2023;9:455–6. https://doi.org/10.1001/jamaoncol.2022.7170.</cite> [<a href="https://doi.org/10.1001/jamaoncol.2022.7170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36757717/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-188">
<span class="label">188.</span><cite>Patel VN, Kaelber DC. Using aggregated, de-identified electronic health record data for multivariate pharmacosurveillance: a case study of azathioprine. J Biomed Inform 2014;52:36–42. https://doi.org/10.1016/j.jbi.2013.10.009.</cite> [<a href="https://doi.org/10.1016/j.jbi.2013.10.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4002671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24177317/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-189">
<span class="label">189.</span><cite>
Trifirò G, Coloma PM, Rijnbeek PR, Romio S, Mosseveld B, Weibel D,
<em>et al.</em>
Combining multiple healthcare databases for postmarketing drug and vaccine safety surveillance: why and how? J Intern Med 2014;275:551–61. https://doi.org/10.1111/joim.12159.
</cite> [<a href="https://doi.org/10.1111/joim.12159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24635221/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-190">
<span class="label">190.</span><cite>Koutkias VG, Jaulent M-C. Computational approaches for pharmacovigilance signal detection: toward integrated and semantically-enriched frameworks. Drug Saf 2015;38:219–32. https://doi.org/10.1007/s40264-015-0278-8.</cite> [<a href="https://doi.org/10.1007/s40264-015-0278-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4374117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25749722/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-191">
<span class="label">191.</span><cite>Marsolo K, Margolis PA, Forrest CB, Colletti RB, Hutton JJ. A Digital Architecture for a Network-Based Learning Health System: Integrating Chronic Care Management, Quality Improvement, and Research. EGEMS Wash DC 2015;3:1168. https://doi.org/10.13063/2327-9214.1168.</cite> [<a href="https://doi.org/10.13063/2327-9214.1168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4562738/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26357665/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-192">
<span class="label">192.</span><cite>Harle CA, Lipori G, Hurley RW. Collecting, Integrating, and Disseminating Patient-Reported Outcomes for Research in a Learning Healthcare System. EGEMS Wash DC 2016;4:1240. https://doi.org/10.13063/2327-9214.1240.</cite> [<a href="https://doi.org/10.13063/2327-9214.1240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4975567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27563683/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-193">
<span class="label">193.</span><cite>
Suchard MA, Schuemie MJ, Krumholz HM, You SC, Chen R, Pratt N,
<em>et al.</em>
Comprehensive comparative effectiveness and safety of first-line antihypertensive drug classes: a systematic, multinational, large-scale analysis. Lancet Lond Engl 2019;394:1816–26. https://doi.org/10.1016/S0140-6736(19)32317-7.
</cite> [<a href="https://doi.org/10.1016/S0140-6736(19)32317-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6924620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31668726/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-194">
<span class="label">194.</span><cite>Platt JE, Raj M, Wienroth M. An Analysis of the Learning Health System in Its First Decade in Practice: Scoping Review. J Med Internet Res 2020;22:e17026. https://doi.org/10.2196/17026.</cite> [<a href="https://doi.org/10.2196/17026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7118548/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32191214/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-195">
<span class="label">195.</span><cite>
Geva A, Stedman JP, Manzi SF, Lin C, Savova GK, Avillach P,
<em>et al.</em>
Adverse drug event presentation and tracking (ADEPT): semiautomated, high throughput pharmacovigilance using real-world data. JAMIA Open 2020;3:413–21. https://doi.org/10.1093/jamiaopen/ooaa031.
</cite> [<a href="https://doi.org/10.1093/jamiaopen/ooaa031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7660953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33215076/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-196">
<span class="label">196.</span><cite>Hartley DM, Seid M. Collaborative learning health systems: Science and practice. Learn Health Syst 2021;5:e10286. https://doi.org/10.1002/lrh2.10286.</cite> [<a href="https://doi.org/10.1002/lrh2.10286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8278439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34277947/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-197">
<span class="label">197.</span><cite>
Xu H, Aldrich MC, Chen Q, Liu H, Peterson NB, Dai Q,
<em>et al.</em>
Validating drug repurposing signals using electronic health records: a case study of metformin associated with reduced cancer mortality. J Am Med Inform Assoc JAMIA 2015;22:179–91. https://doi.org/10.1136/amiajnl-2014-002649.
</cite> [<a href="https://doi.org/10.1136/amiajnl-2014-002649" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4433365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25053577/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-198">
<span class="label">198.</span><cite>Park Y, Jackson GP, Foreman MA, Gruen D, Hu J, Das AK. Evaluating artificial intelligence in medicine: phases of clinical research. JAMIA Open 2020;3:326–31. https://doi.org/10.1093/jamiaopen/ooaa033.</cite> [<a href="https://doi.org/10.1093/jamiaopen/ooaa033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7660958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33215066/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-199">
<span class="label">199.</span><cite>
Mullen CG, Houlihan JY, Stroo M, Deeter CE, Freel SA, Padget AM,
<em>et al.</em>
Leveraging retooled clinical research infrastructure for Clinical Research Management System implementation at a large Academic Medical Center. J Clin Transl Sci 2023;7:e127. https://doi.org/10.1017/cts.2023.550.
</cite> [<a href="https://doi.org/10.1017/cts.2023.550" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10260330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37313387/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-200">
<span class="label">200.</span><cite>
Kwee SA, Wong LL, Ludema C, Deng CK, Taira D, Seto T,
<em>et al.</em>
Target Trial Emulation: A Design Tool for Cancer Clinical Trials. JCO Clin Cancer Inform 2023;7:e2200140. https://doi.org/10.1200/CCI.22.00140.
</cite> [<a href="https://doi.org/10.1200/CCI.22.00140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10166475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36608311/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-201">
<span class="label">201.</span><cite>
Wang SV, Schneeweiss S, RCT-DUPLICATE Initiative, Franklin JM, Desai RJ, Feldman W,
<em>et al.</em>
Emulation of Randomized Clinical Trials With Nonrandomized Database Analyses: Results of 32 Clinical Trials. JAMA 2023;329:1376–85. https://doi.org/10.1001/jama.2023.4221.
</cite> [<a href="https://doi.org/10.1001/jama.2023.4221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10130954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37097356/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-202">
<span class="label">202.</span><cite>
Ganzinger M, Blumenstock M, Fürstberger A, Greulich L, Kestler HA, Marschollek M,
<em>et al.</em>
Federated electronic data capture (fEDC): Architecture and prototype. J Biomed Inform 2023;138:104280. https://doi.org/10.1016/j.jbi.2023.104280.
</cite> [<a href="https://doi.org/10.1016/j.jbi.2023.104280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36623781/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-203">
<span class="label">203.</span><cite>Gama F, Tyskbo D, Nygren J, Barlow J, Reed J, Svedberg P. Implementation Frameworks for Artificial Intelligence Translation Into Health Care Practice: Scoping Review. J Med Internet Res 2022;24:e32215. https://doi.org/10.2196/32215.</cite> [<a href="https://doi.org/10.2196/32215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8832266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35084349/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-204">
<span class="label">204.</span><cite>Crossnohere NL, Elsaid M, Paskett J, Bose-Brill S, Bridges JFP. Guidelines for Artificial Intelligence in Medicine: Literature Review and Content Analysis of Frameworks. J Med Internet Res 2022;24:e36823. https://doi.org/10.2196/36823.</cite> [<a href="https://doi.org/10.2196/36823" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9459836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36006692/" class="usa-link">PubMed</a>]</li>
<li id="ORdaniel-205">
<span class="label">205.</span><cite>
Cruz Rivera S, Liu X, Chan A-W, Denniston AK, Calvert MJ, SPIRIT-AI and CONSORT-AI Working Group,
<em>et al.</em>
Guidelines for clinical trial protocols for interventions involving artificial intelligence: the SPIRIT-AI extension. Nat Med 2020;26:1351–63. https://doi.org/10.1038/s41591-020-1037-7.
</cite> [<a href="https://doi.org/10.1038/s41591-020-1037-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7598944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32908284/" class="usa-link">PubMed</a>]</li>
</ul></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Yearbook of Medical Informatics are provided here courtesy of <strong>Thieme Medical Publishers</strong></p></footer></section></article>



                    </main>
                </div>
            </div>
        </div>





<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">

               <li>
                     <a
                             href="https://doi.org/10.1055/s-0044-1800732"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>


               <li>
                    <a
                            href="pdf/10-1055-s-0044-1800732.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.2 MB)</span>
                    </a>
               </li>


               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12020646/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12020646/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>

               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12020646%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>


<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12020646/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12020646/">
            <h2 class="margin-top-0">RESOURCES</h2>

                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12020646/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"

                                data-source-url="/resources/similar-article-links/40199298/"

                    >

                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12020646/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"

                                data-source-url="/resources/cited-by-links/40199298/"

                    >

                    </div>

                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12020646/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12020646/"
                        >
                        </div>


                </div>

        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>




<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>



<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>


<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >

      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>

      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>

      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>

      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>

  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">



<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="zvRrTfmkNK2kiqxFUPvklofh74uZmvKlG4IQ7knamRKcMV59B5aG4M8sVBRKK2X3">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>



      </div>
    </div>
  </div>







<footer class="ncbi-footer ncbi-dark-background " >

        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">

                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>



                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>



                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>



                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>



                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>

            </div>
        </div>


    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">

                <div class="ncbi-footer__column">

                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>


                    <div class="ncbi-footer__circled-icons-list">

                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>



                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>



                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>

                    </div>
                </div>



                <address class="ncbi-footer__address ncbi-footer__column">

        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>

                </address>



                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">

                        <li class="ncbi-footer__vertical-list-item">










    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >



            Web Policies



    </a>


                        </li>

                        <li class="ncbi-footer__vertical-list-item">










    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >



            FOIA



    </a>


                        </li>

                        <li class="ncbi-footer__vertical-list-item">










    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >



            HHS Vulnerability Disclosure



    </a>


                        </li>

                </ul>



                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">

                        <li class="ncbi-footer__vertical-list-item">










    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >



            Help



    </a>


                        </li>

                        <li class="ncbi-footer__vertical-list-item">










    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >



            Accessibility



    </a>


                        </li>

                        <li class="ncbi-footer__vertical-list-item">










    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >



            Careers



    </a>


                        </li>

                </ul>

        </div>


            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">

                        <li class="ncbi-footer__bottom-list-item">










    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >



            NLM



    </a>


                        </li>

                        <li class="ncbi-footer__bottom-list-item">










    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >



            NIH



    </a>


                        </li>

                        <li class="ncbi-footer__bottom-list-item">










    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >



            HHS



    </a>


                        </li>

                        <li class="ncbi-footer__bottom-list-item">










    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >



            USA.gov



    </a>


                        </li>

                </ul>
            </div>

    </div>
</footer>







  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>





<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>








    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',

        staticEndpoint: '/static/',

        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>

    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>



    </body>
</html>
